[
    "y/core -Inumpy/core/src/npymath -Inumpy/core/src/multiarray -Inump\n             y/core/src/umath -Inumpy/core/src/npysort -I/p/vast1/koparasy/BUIL\n             D/spack_cluster_installer/quartz/py-petsc4py/spack_env_dir/linux-r\n             hel7-broadwell/gcc-9.3.1/python-3.9.9-3o76m4utiyrsvuxwx2ytcv7sd3g3\n             5si6/include/python3.9 -c'\n     869     gcc: _configtest.c\n870     _configtest.c:3: warning: ignoring #pragma clang diagnostic [-Wunk\n             nown-pragmas]\n     871         3 | #pragma clang diagnostic error \"-Wattributes\"\n     872           |\n     873     _configtest.c: In function '__declspec':\n  >> 874     _configtest.c:5:24: error: expected declaration specifiers before\n             'foo'",
    "875         5 | int __declspec(thread) foo;\n     876           |                        ^~~\n  >> 877     _configtest.c:9:1: error: expected '=', ',', ';', 'asm' or '__attr\n             ibute__' before '{' token\n     878         9 | {\n     879           | ^\n880     _configtest.c:5:5: warning: type of 'thread' defaults to 'int' [-W\n             implicit-int]\n     881         5 | int __declspec(thread) foo;\n     882           |     ^~~~~~~~~~\n  >> 883     _configtest.c:11: error: expected '{' at end of input\n     884        11 | }\n     885           |",
    "implicit-int]\n     881         5 | int __declspec(thread) foo;\n     882           |     ^~~~~~~~~~\n  >> 883     _configtest.c:11: error: expected '{' at end of input\n     884        11 | }\n     885           |\n886     _configtest.c:11: warning: control reaches end of non-void functio\n             n [-Wreturn-type]\n     887        11 | }\n     888           |\n     889     _configtest.c:3: warning: ignoring #pragma clang diagnostic [-Wunk\n             nown-pragmas]\n     890         3 | #pragma clang diagnostic error \"-Wattributes\"",
    "884        11 | }\n     885           |\n     886     _configtest.c:11: warning: control reaches end of non-void functio\n             n [-Wreturn-type]\n     887        11 | }\n     888           |\n889     _configtest.c:3: warning: ignoring #pragma clang diagnostic [-Wunk\n             nown-pragmas]\n     890         3 | #pragma clang diagnostic error \"-Wattributes\"\n     891           |\n     892     _configtest.c: In function '__declspec':\n  >> 893     _configtest.c:5:24: error: expected declaration specifiers before\n             'foo'",
    "894         5 | int __declspec(thread) foo;\n     895           |                        ^~~\n  >> 896     _configtest.c:9:1: error: expected '=', ',', ';', 'asm' or '__attr\n             ibute__' before '{' token\n     897         9 | {\n     898           | ^\n899     _configtest.c:5:5: warning: type of 'thread' defaults to 'int' [-W\n             implicit-int]\n     900         5 | int __declspec(thread) foo;\n     901           |     ^~~~~~~~~~\n  >> 902     _configtest.c:11: error: expected '{' at end of input\n     903        11 | }\n     904           |",
    "implicit-int]\n     900         5 | int __declspec(thread) foo;\n     901           |     ^~~~~~~~~~\n  >> 902     _configtest.c:11: error: expected '{' at end of input\n     903        11 | }\n     904           |\n905     _configtest.c:11: warning: control reaches end of non-void functio\n             n [-Wreturn-type]\n     906        11 | }\n     907           |\n     908     failure.\n\n     ...",
    ">> 4425    numpy/random/mtrand/mtrand.c:43644:33: error: 'PyTypeObject' {aka\n             'struct _typeobject'} has no member named 'tp_print'\n     4426    43644 |   __pyx_type_6mtrand_RandomState.tp_print = 0;\n     4427          |                                 ^\n     4428    numpy/random/mtrand/mtrand.c: In function '__Pyx_ParseOptionalKeyw\n             ords':\n4429    numpy/random/mtrand/mtrand.c:45175:21: warning: '_PyUnicode_get_ws\n             tr_length' is deprecated [-Wdeprecated-declarations]\n     4430    45175 |                     (PyUnicode_GET_SIZE(**name) != PyUnico\n             de_GET_SIZE(key)) ? 1 :\n     4431          |                     ^\n\n     ...",
    ">> 4601    numpy/random/mtrand/mtrand.c:43644:33: error: 'PyTypeObject' {aka\n             'struct _typeobject'} has no member named 'tp_print'\n     4602    43644 |   __pyx_type_6mtrand_RandomState.tp_print = 0;\n     4603          |                                 ^\n     4604    numpy/random/mtrand/mtrand.c: In function '__Pyx_ParseOptionalKeyw\n             ords':\n4605    numpy/random/mtrand/mtrand.c:45175:21: warning: '_PyUnicode_get_ws\n             tr_length' is deprecated [-Wdeprecated-declarations]\n     4606    45175 |                     (PyUnicode_GET_SIZE(**name) != PyUnico\n             de_GET_SIZE(key)) ? 1 :\n     4607          |                     ^\n\n     ...",
    "y/core -Inumpy/core/src/npymath -Inumpy/core/src/multiarray -Inump\n             y/core/src/umath -Inumpy/core/src/npysort -I/p/vast1/koparasy/BUIL\n             D/spack_cluster_installer/quartz/py-libensemble/spack_env_dir/linu\n             x-rhel7-broadwell/gcc-9.3.1/python-3.9.9-3o76m4utiyrsvuxwx2ytcv7sd\n             3g35si6/include/python3.9 -c'\n     869     gcc: _configtest.c\n870     _configtest.c:3: warning: ignoring #pragma clang diagnostic [-Wunk\n             nown-pragmas]\n     871         3 | #pragma clang diagnostic error \"-Wattributes\"\n     872           |\n     873     _configtest.c: In function '__declspec':\n  >> 874     _configtest.c:5:24: error: expected declaration specifiers before\n             'foo'",
    "875         5 | int __declspec(thread) foo;\n     876           |                        ^~~\n  >> 877     _configtest.c:9:1: error: expected '=', ',', ';', 'asm' or '__attr\n             ibute__' before '{' token\n     878         9 | {\n     879           | ^\n880     _configtest.c:5:5: warning: type of 'thread' defaults to 'int' [-W\n             implicit-int]\n     881         5 | int __declspec(thread) foo;\n     882           |     ^~~~~~~~~~\n  >> 883     _configtest.c:11: error: expected '{' at end of input\n     884        11 | }\n     885           |",
    "implicit-int]\n     881         5 | int __declspec(thread) foo;\n     882           |     ^~~~~~~~~~\n  >> 883     _configtest.c:11: error: expected '{' at end of input\n     884        11 | }\n     885           |\n886     _configtest.c:11: warning: control reaches end of non-void functio\n             n [-Wreturn-type]\n     887        11 | }\n     888           |\n     889     _configtest.c:3: warning: ignoring #pragma clang diagnostic [-Wunk\n             nown-pragmas]\n     890         3 | #pragma clang diagnostic error \"-Wattributes\"",
    "884        11 | }\n     885           |\n     886     _configtest.c:11: warning: control reaches end of non-void functio\n             n [-Wreturn-type]\n     887        11 | }\n     888           |\n889     _configtest.c:3: warning: ignoring #pragma clang diagnostic [-Wunk\n             nown-pragmas]\n     890         3 | #pragma clang diagnostic error \"-Wattributes\"\n     891           |\n     892     _configtest.c: In function '__declspec':\n  >> 893     _configtest.c:5:24: error: expected declaration specifiers before\n             'foo'",
    "894         5 | int __declspec(thread) foo;\n     895           |                        ^~~\n  >> 896     _configtest.c:9:1: error: expected '=', ',', ';', 'asm' or '__attr\n             ibute__' before '{' token\n     897         9 | {\n     898           | ^\n899     _configtest.c:5:5: warning: type of 'thread' defaults to 'int' [-W\n             implicit-int]\n     900         5 | int __declspec(thread) foo;\n     901           |     ^~~~~~~~~~\n  >> 902     _configtest.c:11: error: expected '{' at end of input\n     903        11 | }\n     904           |",
    "implicit-int]\n     900         5 | int __declspec(thread) foo;\n     901           |     ^~~~~~~~~~\n  >> 902     _configtest.c:11: error: expected '{' at end of input\n     903        11 | }\n     904           |\n905     _configtest.c:11: warning: control reaches end of non-void functio\n             n [-Wreturn-type]\n     906        11 | }\n     907           |\n     908     failure.\n\n     ...",
    ">> 4425    numpy/random/mtrand/mtrand.c:43644:33: error: 'PyTypeObject' {aka\n             'struct _typeobject'} has no member named 'tp_print'\n     4426    43644 |   __pyx_type_6mtrand_RandomState.tp_print = 0;\n     4427          |                                 ^\n     4428    numpy/random/mtrand/mtrand.c: In function '__Pyx_ParseOptionalKeyw\n             ords':\n4429    numpy/random/mtrand/mtrand.c:45175:21: warning: '_PyUnicode_get_ws\n             tr_length' is deprecated [-Wdeprecated-declarations]\n     4430    45175 |                     (PyUnicode_GET_SIZE(**name) != PyUnico\n             de_GET_SIZE(key)) ? 1 :\n     4431          |                     ^\n\n     ...",
    ">> 4601    numpy/random/mtrand/mtrand.c:43644:33: error: 'PyTypeObject' {aka\n             'struct _typeobject'} has no member named 'tp_print'\n     4602    43644 |   __pyx_type_6mtrand_RandomState.tp_print = 0;\n     4603          |                                 ^\n     4604    numpy/random/mtrand/mtrand.c: In function '__Pyx_ParseOptionalKeyw\n             ords':\n4605    numpy/random/mtrand/mtrand.c:45175:21: warning: '_PyUnicode_get_ws\n             tr_length' is deprecated [-Wdeprecated-declarations]\n     4606    45175 |                     (PyUnicode_GET_SIZE(**name) != PyUnico\n             de_GET_SIZE(key)) ? 1 :\n     4607          |                     ^\n\n     ...",
    "st.o -o _configtest\n     115     /usr/tce/packages/gcc/gcc-9.3.1/rh/usr/bin/../libexec/gcc/x86_64-r\n             edhat-linux/9/ld: _configtest.o: in function `main':\n  >> 116     /tmp/koparasy/spack-stage/spack-stage-py-numpy-1.14.1-2d2rt5cmgzkd\n             7ib5m3j7vvhqeeky7wau/spack-src/_configtest.c:6: undefined referenc\n             e to `exp'\n>> 117     collect2: error: ld returned 1 exit status\n     118     /usr/tce/packages/gcc/gcc-9.3.1/rh/usr/bin/../libexec/gcc/x86_64-r\n             edhat-linux/9/ld: _configtest.o: in function `main':\n  >> 119     /tmp/koparasy/spack-stage/spack-stage-py-numpy-1.14.1-2d2rt5cmgzkd\n             7ib5m3j7vvhqeeky7wau/spack-src/_configtest.c:6: undefined referenc\n             e to `exp'\n  >> 120     collect2: error: ld returned 1 exit status",
    ">> 117     collect2: error: ld returned 1 exit status\n     118     /usr/tce/packages/gcc/gcc-9.3.1/rh/usr/bin/../libexec/gcc/x86_64-r\n             edhat-linux/9/ld: _configtest.o: in function `main':\n  >> 119     /tmp/koparasy/spack-stage/spack-stage-py-numpy-1.14.1-2d2rt5cmgzkd\n             7ib5m3j7vvhqeeky7wau/spack-src/_configtest.c:6: undefined referenc\n             e to `exp'\n>> 120     collect2: error: ld returned 1 exit status\n     121     failure.\n     122     removing: _configtest.c _configtest.o\n     123     C compiler: /p/vast1/koparasy/BUILD/head/spack/lib/spack/env/gcc/g\n             cc -Wno-unused-result -Wsign-compare -DNDEBUG -g -fwrapv -O3 -Wall\n              -fPIC -fPIC\n     124",
    "y/core -Inumpy/core/src/npymath -Inumpy/core/src/multiarray -Inump\n             y/core/src/umath -Inumpy/core/src/npysort -I/p/vast1/koparasy/BUIL\n             D/spack_cluster_installer/quartz/py-libensemble/spack_env_dir/linu\n             x-rhel7-broadwell/gcc-9.3.1/python-3.9.9-3o76m4utiyrsvuxwx2ytcv7sd\n             3g35si6/include/python3.9 -c'\n     173     gcc: _configtest.c\n>> 174     _configtest.c:1:10: fatal error: sys/endian.h: No such file or dir\n             ectory\n     175         1 | #include <sys/endian.h>\n     176           |          ^~~~~~~~~~~~~~\n     177     compilation terminated.\n  >> 178     _configtest.c:1:10: fatal error: sys/endian.h: No such file or dir\n             ectory",
    "173     gcc: _configtest.c\n  >> 174     _configtest.c:1:10: fatal error: sys/endian.h: No such file or dir\n             ectory\n     175         1 | #include <sys/endian.h>\n     176           |          ^~~~~~~~~~~~~~\n     177     compilation terminated.\n>> 178     _configtest.c:1:10: fatal error: sys/endian.h: No such file or dir\n             ectory\n     179         1 | #include <sys/endian.h>\n     180           |          ^~~~~~~~~~~~~~\n     181     compilation terminated.\n     182     failure.\n     183     removing: _configtest.c _configtest.o",
    "y/core/src/umath -Inumpy/core/src/npysort -I/p/vast1/koparasy/BUIL\n             D/spack_cluster_installer/quartz/py-libensemble/spack_env_dir/linu\n             x-rhel7-broadwell/gcc-9.3.1/python-3.9.9-3o76m4utiyrsvuxwx2ytcv7sd\n             3g35si6/include/python3.9 -c'\n     289     gcc: _configtest.c\n     290     _configtest.c: In function 'main':\n>> 291     _configtest.c:7:12: error: 'SIZEOF_LONGDOUBLE' undeclared (first u\n             se in this function); did you mean 'SIZEOF_LONG_DOUBLE'?\n     292         7 |     (void) SIZEOF_LONGDOUBLE;\n     293           |            ^~~~~~~~~~~~~~~~~\n     294           |            SIZEOF_LONG_DOUBLE\n     295     _configtest.c:7:12: note: each undeclared identifier is reported o\n             nly once for each function it appears in",
    "292         7 |     (void) SIZEOF_LONGDOUBLE;\n     293           |            ^~~~~~~~~~~~~~~~~\n     294           |            SIZEOF_LONG_DOUBLE\n     295     _configtest.c:7:12: note: each undeclared identifier is reported o\n             nly once for each function it appears in\n     296     _configtest.c: In function 'main':\n>> 297     _configtest.c:7:12: error: 'SIZEOF_LONGDOUBLE' undeclared (first u\n             se in this function); did you mean 'SIZEOF_LONG_DOUBLE'?\n     298         7 |     (void) SIZEOF_LONGDOUBLE;\n     299           |            ^~~~~~~~~~~~~~~~~\n     300           |            SIZEOF_LONG_DOUBLE\n     301     _configtest.c:7:12: note: each undeclared identifier is reported o\n             nly once for each function it appears in",
    "st.o -o _configtest\n     435     /usr/tce/packages/gcc/gcc-9.3.1/rh/usr/bin/../libexec/gcc/x86_64-r\n             edhat-linux/9/ld: _configtest.o: in function `main':\n  >> 436     /tmp/koparasy/spack-stage/spack-stage-py-numpy-1.14.1-2d2rt5cmgzkd\n             7ib5m3j7vvhqeeky7wau/spack-src/_configtest.c:6: undefined referenc\n             e to `exp'\n>> 437     collect2: error: ld returned 1 exit status\n     438     /usr/tce/packages/gcc/gcc-9.3.1/rh/usr/bin/../libexec/gcc/x86_64-r\n             edhat-linux/9/ld: _configtest.o: in function `main':\n  >> 439     /tmp/koparasy/spack-stage/spack-stage-py-numpy-1.14.1-2d2rt5cmgzkd\n             7ib5m3j7vvhqeeky7wau/spack-src/_configtest.c:6: undefined referenc\n             e to `exp'\n  >> 440     collect2: error: ld returned 1 exit status",
    ">> 437     collect2: error: ld returned 1 exit status\n     438     /usr/tce/packages/gcc/gcc-9.3.1/rh/usr/bin/../libexec/gcc/x86_64-r\n             edhat-linux/9/ld: _configtest.o: in function `main':\n  >> 439     /tmp/koparasy/spack-stage/spack-stage-py-numpy-1.14.1-2d2rt5cmgzkd\n             7ib5m3j7vvhqeeky7wau/spack-src/_configtest.c:6: undefined referenc\n             e to `exp'\n>> 440     collect2: error: ld returned 1 exit status\n     441     failure.\n     442     removing: _configtest.c _configtest.o\n     443     C compiler: /p/vast1/koparasy/BUILD/head/spack/lib/spack/env/gcc/g\n             cc -Wno-unused-result -Wsign-compare -DNDEBUG -g -fwrapv -O3 -Wall\n              -fPIC -fPIC\n     444",
    "869     gcc: _configtest.c\n     870     _configtest.c:3: warning: ignoring #pragma clang diagnostic [-Wunk\n             nown-pragmas]\n     871         3 | #pragma clang diagnostic error \"-Wattributes\"\n     872           |\n     873     _configtest.c: In function '__declspec':\n>> 874     _configtest.c:5:24: error: expected declaration specifiers before\n             'foo'\n     875         5 | int __declspec(thread) foo;\n     876           |                        ^~~\n  >> 877     _configtest.c:9:1: error: expected '=', ',', ';', 'asm' or '__attr\n             ibute__' before '{' token\n     878         9 | {",
    "872           |\n     873     _configtest.c: In function '__declspec':\n  >> 874     _configtest.c:5:24: error: expected declaration specifiers before\n             'foo'\n     875         5 | int __declspec(thread) foo;\n     876           |                        ^~~\n>> 877     _configtest.c:9:1: error: expected '=', ',', ';', 'asm' or '__attr\n             ibute__' before '{' token\n     878         9 | {\n     879           | ^\n     880     _configtest.c:5:5: warning: type of 'thread' defaults to 'int' [-W\n             implicit-int]\n     881         5 | int __declspec(thread) foo;",
    "878         9 | {\n     879           | ^\n     880     _configtest.c:5:5: warning: type of 'thread' defaults to 'int' [-W\n             implicit-int]\n     881         5 | int __declspec(thread) foo;\n     882           |     ^~~~~~~~~~\n>> 883     _configtest.c:11: error: expected '{' at end of input\n     884        11 | }\n     885           |\n     886     _configtest.c:11: warning: control reaches end of non-void functio\n             n [-Wreturn-type]\n     887        11 | }\n     888           |",
    "888           |\n     889     _configtest.c:3: warning: ignoring #pragma clang diagnostic [-Wunk\n             nown-pragmas]\n     890         3 | #pragma clang diagnostic error \"-Wattributes\"\n     891           |\n     892     _configtest.c: In function '__declspec':\n>> 893     _configtest.c:5:24: error: expected declaration specifiers before\n             'foo'\n     894         5 | int __declspec(thread) foo;\n     895           |                        ^~~\n  >> 896     _configtest.c:9:1: error: expected '=', ',', ';', 'asm' or '__attr\n             ibute__' before '{' token\n     897         9 | {",
    "891           |\n     892     _configtest.c: In function '__declspec':\n  >> 893     _configtest.c:5:24: error: expected declaration specifiers before\n             'foo'\n     894         5 | int __declspec(thread) foo;\n     895           |                        ^~~\n>> 896     _configtest.c:9:1: error: expected '=', ',', ';', 'asm' or '__attr\n             ibute__' before '{' token\n     897         9 | {\n     898           | ^\n     899     _configtest.c:5:5: warning: type of 'thread' defaults to 'int' [-W\n             implicit-int]\n     900         5 | int __declspec(thread) foo;",
    "897         9 | {\n     898           | ^\n     899     _configtest.c:5:5: warning: type of 'thread' defaults to 'int' [-W\n             implicit-int]\n     900         5 | int __declspec(thread) foo;\n     901           |     ^~~~~~~~~~\n>> 902     _configtest.c:11: error: expected '{' at end of input\n     903        11 | }\n     904           |\n     905     _configtest.c:11: warning: control reaches end of non-void functio\n             n [-Wreturn-type]\n     906        11 | }\n     907           |",
    "y/core/src/umath -Inumpy/core/src/npysort -I/p/vast1/koparasy/BUIL\n             D/spack_cluster_installer/quartz/py-libensemble/spack_env_dir/linu\n             x-rhel7-broadwell/gcc-9.3.1/python-3.9.9-3o76m4utiyrsvuxwx2ytcv7sd\n             3g35si6/include/python3.9 -c'\n     1235    gcc: _configtest.c\n     1236    _configtest.c: In function 'main':\n>> 1237    _configtest.c:8:12: error: 'HAVE_DECL_SIGNBIT' undeclared (first u\n             se in this function); did you mean 'HAVE_DECL_ISNAN'?\n     1238        8 |     (void) HAVE_DECL_SIGNBIT;\n     1239          |            ^~~~~~~~~~~~~~~~~\n     1240          |            HAVE_DECL_ISNAN\n     1241    _configtest.c:8:12: note: each undeclared identifier is reported o\n             nly once for each function it appears in",
    "1238        8 |     (void) HAVE_DECL_SIGNBIT;\n     1239          |            ^~~~~~~~~~~~~~~~~\n     1240          |            HAVE_DECL_ISNAN\n     1241    _configtest.c:8:12: note: each undeclared identifier is reported o\n             nly once for each function it appears in\n     1242    _configtest.c: In function 'main':\n>> 1243    _configtest.c:8:12: error: 'HAVE_DECL_SIGNBIT' undeclared (first u\n             se in this function); did you mean 'HAVE_DECL_ISNAN'?\n     1244        8 |     (void) HAVE_DECL_SIGNBIT;\n     1245          |            ^~~~~~~~~~~~~~~~~\n     1246          |            HAVE_DECL_ISNAN\n     1247    _configtest.c:8:12: note: each undeclared identifier is reported o\n             nly once for each function it appears in",
    "st.o -o _configtest\n     1829    /usr/tce/packages/gcc/gcc-9.3.1/rh/usr/bin/../libexec/gcc/x86_64-r\n             edhat-linux/9/ld: _configtest.o: in function `main':\n  >> 1830    /tmp/koparasy/spack-stage/spack-stage-py-numpy-1.14.1-2d2rt5cmgzkd\n             7ib5m3j7vvhqeeky7wau/spack-src/_configtest.c:6: undefined referenc\n             e to `exp'\n>> 1831    collect2: error: ld returned 1 exit status\n     1832    /usr/tce/packages/gcc/gcc-9.3.1/rh/usr/bin/../libexec/gcc/x86_64-r\n             edhat-linux/9/ld: _configtest.o: in function `main':\n  >> 1833    /tmp/koparasy/spack-stage/spack-stage-py-numpy-1.14.1-2d2rt5cmgzkd\n             7ib5m3j7vvhqeeky7wau/spack-src/_configtest.c:6: undefined referenc\n             e to `exp'\n  >> 1834    collect2: error: ld returned 1 exit status",
    ">> 1831    collect2: error: ld returned 1 exit status\n     1832    /usr/tce/packages/gcc/gcc-9.3.1/rh/usr/bin/../libexec/gcc/x86_64-r\n             edhat-linux/9/ld: _configtest.o: in function `main':\n  >> 1833    /tmp/koparasy/spack-stage/spack-stage-py-numpy-1.14.1-2d2rt5cmgzkd\n             7ib5m3j7vvhqeeky7wau/spack-src/_configtest.c:6: undefined referenc\n             e to `exp'\n>> 1834    collect2: error: ld returned 1 exit status\n     1835    failure.\n     1836    removing: _configtest.c _configtest.o\n     1837    C compiler: /p/vast1/koparasy/BUILD/head/spack/lib/spack/env/gcc/g\n             cc -Wno-unused-result -Wsign-compare -DNDEBUG -g -fwrapv -O3 -Wall\n              -fPIC -fPIC\n     1838",
    "c/npymath -Ibuild/src.linux-x86_64-3.9/numpy/core/src/private -Ibu\n             ild/src.linux-x86_64-3.9/numpy/core/src/npymath -Ibuild/src.linux-\n             x86_64-3.9/numpy/core/src/private -Ibuild/src.linux-x86_64-3.9/num\n             py/core/src/npymath -c'\n     4423    gcc: numpy/random/mtrand/mtrand.c\n     4424    numpy/random/mtrand/mtrand.c: In function 'PyInit_mtrand':\n>> 4425    numpy/random/mtrand/mtrand.c:43644:33: error: 'PyTypeObject' {aka\n             'struct _typeobject'} has no member named 'tp_print'\n     4426    43644 |   __pyx_type_6mtrand_RandomState.tp_print = 0;\n     4427          |                                 ^\n     4428    numpy/random/mtrand/mtrand.c: In function '__Pyx_ParseOptionalKeyw\n             ords':\n     4429    numpy/random/mtrand/mtrand.c:45175:21: warning: '_PyUnicode_get_ws",
    "6m4utiyrsvuxwx2ytcv7sd3g35si6/include/python3.9/cpython/unicodeobj\n             ect.h:446:26: note: declared here\n     4535      446 | static inline Py_ssize_t _PyUnicode_get_wstr_length(PyObje\n             ct *op) {\n     4536          |                          ^~~~~~~~~~~~~~~~~~~~~~~~~~\n     4537    numpy/random/mtrand/mtrand.c: In function '__Pyx__ExceptionSave':\n>> 4538    numpy/random/mtrand/mtrand.c:45472:21: error: 'PyThreadState' {aka\n              'struct _ts'} has no member named 'exc_type'; did you mean 'curex\n             c_type'?\n     4539    45472 |     *type = tstate->exc_type;\n     4540          |                     ^~~~~~~~\n     4541          |                     curexc_type\n  >> 4542    numpy/random/mtrand/mtrand.c:45473:22: error: 'PyThreadState' {aka",
    ">> 4538    numpy/random/mtrand/mtrand.c:45472:21: error: 'PyThreadState' {aka\n              'struct _ts'} has no member named 'exc_type'; did you mean 'curex\n             c_type'?\n     4539    45472 |     *type = tstate->exc_type;\n     4540          |                     ^~~~~~~~\n     4541          |                     curexc_type\n>> 4542    numpy/random/mtrand/mtrand.c:45473:22: error: 'PyThreadState' {aka\n              'struct _ts'} has no member named 'exc_value'; did you mean 'cure\n             xc_value'?\n     4543    45473 |     *value = tstate->exc_value;\n     4544          |                      ^~~~~~~~~\n     4545          |                      curexc_value\n  >> 4546    numpy/random/mtrand/mtrand.c:45474:19: error: 'PyThreadState' {aka",
    ">> 4542    numpy/random/mtrand/mtrand.c:45473:22: error: 'PyThreadState' {aka\n              'struct _ts'} has no member named 'exc_value'; did you mean 'cure\n             xc_value'?\n     4543    45473 |     *value = tstate->exc_value;\n     4544          |                      ^~~~~~~~~\n     4545          |                      curexc_value\n>> 4546    numpy/random/mtrand/mtrand.c:45474:19: error: 'PyThreadState' {aka\n              'struct _ts'} has no member named 'exc_traceback'; did you mean '\n             curexc_traceback'?\n     4547    45474 |     *tb = tstate->exc_traceback;\n     4548          |                   ^~~~~~~~~~~~~\n     4549          |                   curexc_traceback\n     4550    numpy/random/mtrand/mtrand.c: In function '__Pyx__ExceptionReset':",
    "'struct _ts'} has no member named 'exc_traceback'; did you mean '\n             curexc_traceback'?\n     4547    45474 |     *tb = tstate->exc_traceback;\n     4548          |                   ^~~~~~~~~~~~~\n     4549          |                   curexc_traceback\n     4550    numpy/random/mtrand/mtrand.c: In function '__Pyx__ExceptionReset':\n>> 4551    numpy/random/mtrand/mtrand.c:45481:24: error: 'PyThreadState' {aka\n              'struct _ts'} has no member named 'exc_type'; did you mean 'curex\n             c_type'?\n     4552    45481 |     tmp_type = tstate->exc_type;\n     4553          |                        ^~~~~~~~\n     4554          |                        curexc_type\n  >> 4555    numpy/random/mtrand/mtrand.c:45482:25: error: 'PyThreadState' {aka",
    ">> 4551    numpy/random/mtrand/mtrand.c:45481:24: error: 'PyThreadState' {aka\n              'struct _ts'} has no member named 'exc_type'; did you mean 'curex\n             c_type'?\n     4552    45481 |     tmp_type = tstate->exc_type;\n     4553          |                        ^~~~~~~~\n     4554          |                        curexc_type\n>> 4555    numpy/random/mtrand/mtrand.c:45482:25: error: 'PyThreadState' {aka\n              'struct _ts'} has no member named 'exc_value'; did you mean 'cure\n             xc_value'?\n     4556    45482 |     tmp_value = tstate->exc_value;\n     4557          |                         ^~~~~~~~~\n     4558          |                         curexc_value\n  >> 4559    numpy/random/mtrand/mtrand.c:45483:22: error: 'PyThreadState' {aka",
    ">> 4555    numpy/random/mtrand/mtrand.c:45482:25: error: 'PyThreadState' {aka\n              'struct _ts'} has no member named 'exc_value'; did you mean 'cure\n             xc_value'?\n     4556    45482 |     tmp_value = tstate->exc_value;\n     4557          |                         ^~~~~~~~~\n     4558          |                         curexc_value\n>> 4559    numpy/random/mtrand/mtrand.c:45483:22: error: 'PyThreadState' {aka\n              'struct _ts'} has no member named 'exc_traceback'; did you mean '\n             curexc_traceback'?\n     4560    45483 |     tmp_tb = tstate->exc_traceback;\n     4561          |                      ^~~~~~~~~~~~~\n     4562          |                      curexc_traceback\n  >> 4563    numpy/random/mtrand/mtrand.c:45484:13: error: 'PyThreadState' {aka",
    ">> 4559    numpy/random/mtrand/mtrand.c:45483:22: error: 'PyThreadState' {aka\n              'struct _ts'} has no member named 'exc_traceback'; did you mean '\n             curexc_traceback'?\n     4560    45483 |     tmp_tb = tstate->exc_traceback;\n     4561          |                      ^~~~~~~~~~~~~\n     4562          |                      curexc_traceback\n>> 4563    numpy/random/mtrand/mtrand.c:45484:13: error: 'PyThreadState' {aka\n              'struct _ts'} has no member named 'exc_type'; did you mean 'curex\n             c_type'?\n     4564    45484 |     tstate->exc_type = type;\n     4565          |             ^~~~~~~~\n     4566          |             curexc_type\n  >> 4567    numpy/random/mtrand/mtrand.c:45485:13: error: 'PyThreadState' {aka",
    ">> 4563    numpy/random/mtrand/mtrand.c:45484:13: error: 'PyThreadState' {aka\n              'struct _ts'} has no member named 'exc_type'; did you mean 'curex\n             c_type'?\n     4564    45484 |     tstate->exc_type = type;\n     4565          |             ^~~~~~~~\n     4566          |             curexc_type\n>> 4567    numpy/random/mtrand/mtrand.c:45485:13: error: 'PyThreadState' {aka\n              'struct _ts'} has no member named 'exc_value'; did you mean 'cure\n             xc_value'?\n     4568    45485 |     tstate->exc_value = value;\n     4569          |             ^~~~~~~~~\n     4570          |             curexc_value\n  >> 4571    numpy/random/mtrand/mtrand.c:45486:13: error: 'PyThreadState' {aka",
    ">> 4567    numpy/random/mtrand/mtrand.c:45485:13: error: 'PyThreadState' {aka\n              'struct _ts'} has no member named 'exc_value'; did you mean 'cure\n             xc_value'?\n     4568    45485 |     tstate->exc_value = value;\n     4569          |             ^~~~~~~~~\n     4570          |             curexc_value\n>> 4571    numpy/random/mtrand/mtrand.c:45486:13: error: 'PyThreadState' {aka\n              'struct _ts'} has no member named 'exc_traceback'; did you mean '\n             curexc_traceback'?\n     4572    45486 |     tstate->exc_traceback = tb;\n     4573          |             ^~~~~~~~~~~~~\n     4574          |             curexc_traceback\n     4575    numpy/random/mtrand/mtrand.c: In function '__Pyx__GetException':",
    "'struct _ts'} has no member named 'exc_traceback'; did you mean '\n             curexc_traceback'?\n     4572    45486 |     tstate->exc_traceback = tb;\n     4573          |             ^~~~~~~~~~~~~\n     4574          |             curexc_traceback\n     4575    numpy/random/mtrand/mtrand.c: In function '__Pyx__GetException':\n>> 4576    numpy/random/mtrand/mtrand.c:45541:24: error: 'PyThreadState' {aka\n              'struct _ts'} has no member named 'exc_type'; did you mean 'curex\n             c_type'?\n     4577    45541 |     tmp_type = tstate->exc_type;\n     4578          |                        ^~~~~~~~\n     4579          |                        curexc_type\n  >> 4580    numpy/random/mtrand/mtrand.c:45542:25: error: 'PyThreadState' {aka",
    ">> 4576    numpy/random/mtrand/mtrand.c:45541:24: error: 'PyThreadState' {aka\n              'struct _ts'} has no member named 'exc_type'; did you mean 'curex\n             c_type'?\n     4577    45541 |     tmp_type = tstate->exc_type;\n     4578          |                        ^~~~~~~~\n     4579          |                        curexc_type\n>> 4580    numpy/random/mtrand/mtrand.c:45542:25: error: 'PyThreadState' {aka\n              'struct _ts'} has no member named 'exc_value'; did you mean 'cure\n             xc_value'?\n     4581    45542 |     tmp_value = tstate->exc_value;\n     4582          |                         ^~~~~~~~~\n     4583          |                         curexc_value\n  >> 4584    numpy/random/mtrand/mtrand.c:45543:22: error: 'PyThreadState' {aka",
    ">> 4580    numpy/random/mtrand/mtrand.c:45542:25: error: 'PyThreadState' {aka\n              'struct _ts'} has no member named 'exc_value'; did you mean 'cure\n             xc_value'?\n     4581    45542 |     tmp_value = tstate->exc_value;\n     4582          |                         ^~~~~~~~~\n     4583          |                         curexc_value\n>> 4584    numpy/random/mtrand/mtrand.c:45543:22: error: 'PyThreadState' {aka\n              'struct _ts'} has no member named 'exc_traceback'; did you mean '\n             curexc_traceback'?\n     4585    45543 |     tmp_tb = tstate->exc_traceback;\n     4586          |                      ^~~~~~~~~~~~~\n     4587          |                      curexc_traceback\n  >> 4588    numpy/random/mtrand/mtrand.c:45544:13: error: 'PyThreadState' {aka",
    ">> 4584    numpy/random/mtrand/mtrand.c:45543:22: error: 'PyThreadState' {aka\n              'struct _ts'} has no member named 'exc_traceback'; did you mean '\n             curexc_traceback'?\n     4585    45543 |     tmp_tb = tstate->exc_traceback;\n     4586          |                      ^~~~~~~~~~~~~\n     4587          |                      curexc_traceback\n>> 4588    numpy/random/mtrand/mtrand.c:45544:13: error: 'PyThreadState' {aka\n              'struct _ts'} has no member named 'exc_type'; did you mean 'curex\n             c_type'?\n     4589    45544 |     tstate->exc_type = local_type;\n     4590          |             ^~~~~~~~\n     4591          |             curexc_type\n  >> 4592    numpy/random/mtrand/mtrand.c:45545:13: error: 'PyThreadState' {aka",
    ">> 4588    numpy/random/mtrand/mtrand.c:45544:13: error: 'PyThreadState' {aka\n              'struct _ts'} has no member named 'exc_type'; did you mean 'curex\n             c_type'?\n     4589    45544 |     tstate->exc_type = local_type;\n     4590          |             ^~~~~~~~\n     4591          |             curexc_type\n>> 4592    numpy/random/mtrand/mtrand.c:45545:13: error: 'PyThreadState' {aka\n              'struct _ts'} has no member named 'exc_value'; did you mean 'cure\n             xc_value'?\n     4593    45545 |     tstate->exc_value = local_value;\n     4594          |             ^~~~~~~~~\n     4595          |             curexc_value\n  >> 4596    numpy/random/mtrand/mtrand.c:45546:13: error: 'PyThreadState' {aka",
    ">> 4592    numpy/random/mtrand/mtrand.c:45545:13: error: 'PyThreadState' {aka\n              'struct _ts'} has no member named 'exc_value'; did you mean 'cure\n             xc_value'?\n     4593    45545 |     tstate->exc_value = local_value;\n     4594          |             ^~~~~~~~~\n     4595          |             curexc_value\n>> 4596    numpy/random/mtrand/mtrand.c:45546:13: error: 'PyThreadState' {aka\n              'struct _ts'} has no member named 'exc_traceback'; did you mean '\n             curexc_traceback'?\n     4597    45546 |     tstate->exc_traceback = local_tb;\n     4598          |             ^~~~~~~~~~~~~\n     4599          |             curexc_traceback\n     4600    numpy/random/mtrand/mtrand.c: In function 'PyInit_mtrand':",
    "'struct _ts'} has no member named 'exc_traceback'; did you mean '\n             curexc_traceback'?\n     4597    45546 |     tstate->exc_traceback = local_tb;\n     4598          |             ^~~~~~~~~~~~~\n     4599          |             curexc_traceback\n     4600    numpy/random/mtrand/mtrand.c: In function 'PyInit_mtrand':\n>> 4601    numpy/random/mtrand/mtrand.c:43644:33: error: 'PyTypeObject' {aka\n             'struct _typeobject'} has no member named 'tp_print'\n     4602    43644 |   __pyx_type_6mtrand_RandomState.tp_print = 0;\n     4603          |                                 ^\n     4604    numpy/random/mtrand/mtrand.c: In function '__Pyx_ParseOptionalKeyw\n             ords':\n     4605    numpy/random/mtrand/mtrand.c:45175:21: warning: '_PyUnicode_get_ws",
    "6m4utiyrsvuxwx2ytcv7sd3g35si6/include/python3.9/cpython/unicodeobj\n             ect.h:446:26: note: declared here\n     4711      446 | static inline Py_ssize_t _PyUnicode_get_wstr_length(PyObje\n             ct *op) {\n     4712          |                          ^~~~~~~~~~~~~~~~~~~~~~~~~~\n     4713    numpy/random/mtrand/mtrand.c: In function '__Pyx__ExceptionSave':\n>> 4714    numpy/random/mtrand/mtrand.c:45472:21: error: 'PyThreadState' {aka\n              'struct _ts'} has no member named 'exc_type'; did you mean 'curex\n             c_type'?\n     4715    45472 |     *type = tstate->exc_type;\n     4716          |                     ^~~~~~~~\n     4717          |                     curexc_type\n  >> 4718    numpy/random/mtrand/mtrand.c:45473:22: error: 'PyThreadState' {aka",
    ">> 4714    numpy/random/mtrand/mtrand.c:45472:21: error: 'PyThreadState' {aka\n              'struct _ts'} has no member named 'exc_type'; did you mean 'curex\n             c_type'?\n     4715    45472 |     *type = tstate->exc_type;\n     4716          |                     ^~~~~~~~\n     4717          |                     curexc_type\n>> 4718    numpy/random/mtrand/mtrand.c:45473:22: error: 'PyThreadState' {aka\n              'struct _ts'} has no member named 'exc_value'; did you mean 'cure\n             xc_value'?\n     4719    45473 |     *value = tstate->exc_value;\n     4720          |                      ^~~~~~~~~\n     4721          |                      curexc_value\n  >> 4722    numpy/random/mtrand/mtrand.c:45474:19: error: 'PyThreadState' {aka",
    ">> 4718    numpy/random/mtrand/mtrand.c:45473:22: error: 'PyThreadState' {aka\n              'struct _ts'} has no member named 'exc_value'; did you mean 'cure\n             xc_value'?\n     4719    45473 |     *value = tstate->exc_value;\n     4720          |                      ^~~~~~~~~\n     4721          |                      curexc_value\n>> 4722    numpy/random/mtrand/mtrand.c:45474:19: error: 'PyThreadState' {aka\n              'struct _ts'} has no member named 'exc_traceback'; did you mean '\n             curexc_traceback'?\n     4723    45474 |     *tb = tstate->exc_traceback;\n     4724          |                   ^~~~~~~~~~~~~\n     4725          |                   curexc_traceback\n     4726    numpy/random/mtrand/mtrand.c: In function '__Pyx__ExceptionReset':",
    "'struct _ts'} has no member named 'exc_traceback'; did you mean '\n             curexc_traceback'?\n     4723    45474 |     *tb = tstate->exc_traceback;\n     4724          |                   ^~~~~~~~~~~~~\n     4725          |                   curexc_traceback\n     4726    numpy/random/mtrand/mtrand.c: In function '__Pyx__ExceptionReset':\n>> 4727    numpy/random/mtrand/mtrand.c:45481:24: error: 'PyThreadState' {aka\n              'struct _ts'} has no member named 'exc_type'; did you mean 'curex\n             c_type'?\n     4728    45481 |     tmp_type = tstate->exc_type;\n     4729          |                        ^~~~~~~~\n     4730          |                        curexc_type\n  >> 4731    numpy/random/mtrand/mtrand.c:45482:25: error: 'PyThreadState' {aka",
    ">> 4727    numpy/random/mtrand/mtrand.c:45481:24: error: 'PyThreadState' {aka\n              'struct _ts'} has no member named 'exc_type'; did you mean 'curex\n             c_type'?\n     4728    45481 |     tmp_type = tstate->exc_type;\n     4729          |                        ^~~~~~~~\n     4730          |                        curexc_type\n>> 4731    numpy/random/mtrand/mtrand.c:45482:25: error: 'PyThreadState' {aka\n              'struct _ts'} has no member named 'exc_value'; did you mean 'cure\n             xc_value'?\n     4732    45482 |     tmp_value = tstate->exc_value;\n     4733          |                         ^~~~~~~~~\n     4734          |                         curexc_value\n  >> 4735    numpy/random/mtrand/mtrand.c:45483:22: error: 'PyThreadState' {aka",
    ">> 4731    numpy/random/mtrand/mtrand.c:45482:25: error: 'PyThreadState' {aka\n              'struct _ts'} has no member named 'exc_value'; did you mean 'cure\n             xc_value'?\n     4732    45482 |     tmp_value = tstate->exc_value;\n     4733          |                         ^~~~~~~~~\n     4734          |                         curexc_value\n>> 4735    numpy/random/mtrand/mtrand.c:45483:22: error: 'PyThreadState' {aka\n              'struct _ts'} has no member named 'exc_traceback'; did you mean '\n             curexc_traceback'?\n     4736    45483 |     tmp_tb = tstate->exc_traceback;\n     4737          |                      ^~~~~~~~~~~~~\n     4738          |                      curexc_traceback\n  >> 4739    numpy/random/mtrand/mtrand.c:45484:13: error: 'PyThreadState' {aka",
    ">> 4735    numpy/random/mtrand/mtrand.c:45483:22: error: 'PyThreadState' {aka\n              'struct _ts'} has no member named 'exc_traceback'; did you mean '\n             curexc_traceback'?\n     4736    45483 |     tmp_tb = tstate->exc_traceback;\n     4737          |                      ^~~~~~~~~~~~~\n     4738          |                      curexc_traceback\n>> 4739    numpy/random/mtrand/mtrand.c:45484:13: error: 'PyThreadState' {aka\n              'struct _ts'} has no member named 'exc_type'; did you mean 'curex\n             c_type'?\n     4740    45484 |     tstate->exc_type = type;\n     4741          |             ^~~~~~~~\n     4742          |             curexc_type\n  >> 4743    numpy/random/mtrand/mtrand.c:45485:13: error: 'PyThreadState' {aka",
    ">> 4739    numpy/random/mtrand/mtrand.c:45484:13: error: 'PyThreadState' {aka\n              'struct _ts'} has no member named 'exc_type'; did you mean 'curex\n             c_type'?\n     4740    45484 |     tstate->exc_type = type;\n     4741          |             ^~~~~~~~\n     4742          |             curexc_type\n>> 4743    numpy/random/mtrand/mtrand.c:45485:13: error: 'PyThreadState' {aka\n              'struct _ts'} has no member named 'exc_value'; did you mean 'cure\n             xc_value'?\n     4744    45485 |     tstate->exc_value = value;\n     4745          |             ^~~~~~~~~\n     4746          |             curexc_value\n  >> 4747    numpy/random/mtrand/mtrand.c:45486:13: error: 'PyThreadState' {aka",
    ">> 4743    numpy/random/mtrand/mtrand.c:45485:13: error: 'PyThreadState' {aka\n              'struct _ts'} has no member named 'exc_value'; did you mean 'cure\n             xc_value'?\n     4744    45485 |     tstate->exc_value = value;\n     4745          |             ^~~~~~~~~\n     4746          |             curexc_value\n>> 4747    numpy/random/mtrand/mtrand.c:45486:13: error: 'PyThreadState' {aka\n              'struct _ts'} has no member named 'exc_traceback'; did you mean '\n             curexc_traceback'?\n     4748    45486 |     tstate->exc_traceback = tb;\n     4749          |             ^~~~~~~~~~~~~\n     4750          |             curexc_traceback\n     4751    numpy/random/mtrand/mtrand.c: In function '__Pyx__GetException':",
    "'struct _ts'} has no member named 'exc_traceback'; did you mean '\n             curexc_traceback'?\n     4748    45486 |     tstate->exc_traceback = tb;\n     4749          |             ^~~~~~~~~~~~~\n     4750          |             curexc_traceback\n     4751    numpy/random/mtrand/mtrand.c: In function '__Pyx__GetException':\n>> 4752    numpy/random/mtrand/mtrand.c:45541:24: error: 'PyThreadState' {aka\n              'struct _ts'} has no member named 'exc_type'; did you mean 'curex\n             c_type'?\n     4753    45541 |     tmp_type = tstate->exc_type;\n     4754          |                        ^~~~~~~~\n     4755          |                        curexc_type\n  >> 4756    numpy/random/mtrand/mtrand.c:45542:25: error: 'PyThreadState' {aka",
    ">> 4752    numpy/random/mtrand/mtrand.c:45541:24: error: 'PyThreadState' {aka\n              'struct _ts'} has no member named 'exc_type'; did you mean 'curex\n             c_type'?\n     4753    45541 |     tmp_type = tstate->exc_type;\n     4754          |                        ^~~~~~~~\n     4755          |                        curexc_type\n>> 4756    numpy/random/mtrand/mtrand.c:45542:25: error: 'PyThreadState' {aka\n              'struct _ts'} has no member named 'exc_value'; did you mean 'cure\n             xc_value'?\n     4757    45542 |     tmp_value = tstate->exc_value;\n     4758          |                         ^~~~~~~~~\n     4759          |                         curexc_value\n  >> 4760    numpy/random/mtrand/mtrand.c:45543:22: error: 'PyThreadState' {aka",
    ">> 4756    numpy/random/mtrand/mtrand.c:45542:25: error: 'PyThreadState' {aka\n              'struct _ts'} has no member named 'exc_value'; did you mean 'cure\n             xc_value'?\n     4757    45542 |     tmp_value = tstate->exc_value;\n     4758          |                         ^~~~~~~~~\n     4759          |                         curexc_value\n>> 4760    numpy/random/mtrand/mtrand.c:45543:22: error: 'PyThreadState' {aka\n              'struct _ts'} has no member named 'exc_traceback'; did you mean '\n             curexc_traceback'?\n     4761    45543 |     tmp_tb = tstate->exc_traceback;\n     4762          |                      ^~~~~~~~~~~~~\n     4763          |                      curexc_traceback\n  >> 4764    numpy/random/mtrand/mtrand.c:45544:13: error: 'PyThreadState' {aka",
    ">> 4760    numpy/random/mtrand/mtrand.c:45543:22: error: 'PyThreadState' {aka\n              'struct _ts'} has no member named 'exc_traceback'; did you mean '\n             curexc_traceback'?\n     4761    45543 |     tmp_tb = tstate->exc_traceback;\n     4762          |                      ^~~~~~~~~~~~~\n     4763          |                      curexc_traceback\n>> 4764    numpy/random/mtrand/mtrand.c:45544:13: error: 'PyThreadState' {aka\n              'struct _ts'} has no member named 'exc_type'; did you mean 'curex\n             c_type'?\n     4765    45544 |     tstate->exc_type = local_type;\n     4766          |             ^~~~~~~~\n     4767          |             curexc_type\n  >> 4768    numpy/random/mtrand/mtrand.c:45545:13: error: 'PyThreadState' {aka",
    ">> 4764    numpy/random/mtrand/mtrand.c:45544:13: error: 'PyThreadState' {aka\n              'struct _ts'} has no member named 'exc_type'; did you mean 'curex\n             c_type'?\n     4765    45544 |     tstate->exc_type = local_type;\n     4766          |             ^~~~~~~~\n     4767          |             curexc_type\n>> 4768    numpy/random/mtrand/mtrand.c:45545:13: error: 'PyThreadState' {aka\n              'struct _ts'} has no member named 'exc_value'; did you mean 'cure\n             xc_value'?\n     4769    45545 |     tstate->exc_value = local_value;\n     4770          |             ^~~~~~~~~\n     4771          |             curexc_value\n  >> 4772    numpy/random/mtrand/mtrand.c:45546:13: error: 'PyThreadState' {aka",
    ">> 4768    numpy/random/mtrand/mtrand.c:45545:13: error: 'PyThreadState' {aka\n              'struct _ts'} has no member named 'exc_value'; did you mean 'cure\n             xc_value'?\n     4769    45545 |     tstate->exc_value = local_value;\n     4770          |             ^~~~~~~~~\n     4771          |             curexc_value\n>> 4772    numpy/random/mtrand/mtrand.c:45546:13: error: 'PyThreadState' {aka\n              'struct _ts'} has no member named 'exc_traceback'; did you mean '\n             curexc_traceback'?\n     4773    45546 |     tstate->exc_traceback = local_tb;\n     4774          |             ^~~~~~~~~~~~~\n     4775          |             curexc_traceback\n  >> 4776    error: Command \"/p/vast1/koparasy/BUILD/head/spack/lib/spack/env/g",
    "y/core -Inumpy/core/src/npymath -Inumpy/core/src/multiarray -Inump\n             y/core/src/umath -Inumpy/core/src/npysort -I/p/vast1/koparasy/BUIL\n             D/spack_cluster_installer/quartz/py-warpx/spack_env_dir/linux-rhel\n             7-broadwell/gcc-9.3.1/python-3.9.9-3o76m4utiyrsvuxwx2ytcv7sd3g35si\n             6/include/python3.9 -c'\n     869     gcc: _configtest.c\n870     _configtest.c:3: warning: ignoring #pragma clang diagnostic [-Wunk\n             nown-pragmas]\n     871         3 | #pragma clang diagnostic error \"-Wattributes\"\n     872           |\n     873     _configtest.c: In function '__declspec':\n  >> 874     _configtest.c:5:24: error: expected declaration specifiers before\n             'foo'",
    "875         5 | int __declspec(thread) foo;\n     876           |                        ^~~\n  >> 877     _configtest.c:9:1: error: expected '=', ',', ';', 'asm' or '__attr\n             ibute__' before '{' token\n     878         9 | {\n     879           | ^\n880     _configtest.c:5:5: warning: type of 'thread' defaults to 'int' [-W\n             implicit-int]\n     881         5 | int __declspec(thread) foo;\n     882           |     ^~~~~~~~~~\n  >> 883     _configtest.c:11: error: expected '{' at end of input\n     884        11 | }\n     885           |",
    "implicit-int]\n     881         5 | int __declspec(thread) foo;\n     882           |     ^~~~~~~~~~\n  >> 883     _configtest.c:11: error: expected '{' at end of input\n     884        11 | }\n     885           |\n886     _configtest.c:11: warning: control reaches end of non-void functio\n             n [-Wreturn-type]\n     887        11 | }\n     888           |\n     889     _configtest.c:3: warning: ignoring #pragma clang diagnostic [-Wunk\n             nown-pragmas]\n     890         3 | #pragma clang diagnostic error \"-Wattributes\"",
    "884        11 | }\n     885           |\n     886     _configtest.c:11: warning: control reaches end of non-void functio\n             n [-Wreturn-type]\n     887        11 | }\n     888           |\n889     _configtest.c:3: warning: ignoring #pragma clang diagnostic [-Wunk\n             nown-pragmas]\n     890         3 | #pragma clang diagnostic error \"-Wattributes\"\n     891           |\n     892     _configtest.c: In function '__declspec':\n  >> 893     _configtest.c:5:24: error: expected declaration specifiers before\n             'foo'",
    "894         5 | int __declspec(thread) foo;\n     895           |                        ^~~\n  >> 896     _configtest.c:9:1: error: expected '=', ',', ';', 'asm' or '__attr\n             ibute__' before '{' token\n     897         9 | {\n     898           | ^\n899     _configtest.c:5:5: warning: type of 'thread' defaults to 'int' [-W\n             implicit-int]\n     900         5 | int __declspec(thread) foo;\n     901           |     ^~~~~~~~~~\n  >> 902     _configtest.c:11: error: expected '{' at end of input\n     903        11 | }\n     904           |",
    "implicit-int]\n     900         5 | int __declspec(thread) foo;\n     901           |     ^~~~~~~~~~\n  >> 902     _configtest.c:11: error: expected '{' at end of input\n     903        11 | }\n     904           |\n905     _configtest.c:11: warning: control reaches end of non-void functio\n             n [-Wreturn-type]\n     906        11 | }\n     907           |\n     908     failure.\n\n     ...",
    ">> 4425    numpy/random/mtrand/mtrand.c:43644:33: error: 'PyTypeObject' {aka\n             'struct _typeobject'} has no member named 'tp_print'\n     4426    43644 |   __pyx_type_6mtrand_RandomState.tp_print = 0;\n     4427          |                                 ^\n     4428    numpy/random/mtrand/mtrand.c: In function '__Pyx_ParseOptionalKeyw\n             ords':\n4429    numpy/random/mtrand/mtrand.c:45175:21: warning: '_PyUnicode_get_ws\n             tr_length' is deprecated [-Wdeprecated-declarations]\n     4430    45175 |                     (PyUnicode_GET_SIZE(**name) != PyUnico\n             de_GET_SIZE(key)) ? 1 :\n     4431          |                     ^\n\n     ...",
    ">> 4601    numpy/random/mtrand/mtrand.c:43644:33: error: 'PyTypeObject' {aka\n             'struct _typeobject'} has no member named 'tp_print'\n     4602    43644 |   __pyx_type_6mtrand_RandomState.tp_print = 0;\n     4603          |                                 ^\n     4604    numpy/random/mtrand/mtrand.c: In function '__Pyx_ParseOptionalKeyw\n             ords':\n4605    numpy/random/mtrand/mtrand.c:45175:21: warning: '_PyUnicode_get_ws\n             tr_length' is deprecated [-Wdeprecated-declarations]\n     4606    45175 |                     (PyUnicode_GET_SIZE(**name) != PyUnico\n             de_GET_SIZE(key)) ? 1 :\n     4607          |                     ^\n\n     ...",
    "st.o -o _configtest\n     115     /usr/tce/packages/gcc/gcc-9.3.1/rh/usr/bin/../libexec/gcc/x86_64-r\n             edhat-linux/9/ld: _configtest.o: in function `main':\n  >> 116     /tmp/koparasy/spack-stage/spack-stage-py-numpy-1.14.1-2d2rt5cmgzkd\n             7ib5m3j7vvhqeeky7wau/spack-src/_configtest.c:6: undefined referenc\n             e to `exp'\n>> 117     collect2: error: ld returned 1 exit status\n     118     /usr/tce/packages/gcc/gcc-9.3.1/rh/usr/bin/../libexec/gcc/x86_64-r\n             edhat-linux/9/ld: _configtest.o: in function `main':\n  >> 119     /tmp/koparasy/spack-stage/spack-stage-py-numpy-1.14.1-2d2rt5cmgzkd\n             7ib5m3j7vvhqeeky7wau/spack-src/_configtest.c:6: undefined referenc\n             e to `exp'\n  >> 120     collect2: error: ld returned 1 exit status",
    ">> 117     collect2: error: ld returned 1 exit status\n     118     /usr/tce/packages/gcc/gcc-9.3.1/rh/usr/bin/../libexec/gcc/x86_64-r\n             edhat-linux/9/ld: _configtest.o: in function `main':\n  >> 119     /tmp/koparasy/spack-stage/spack-stage-py-numpy-1.14.1-2d2rt5cmgzkd\n             7ib5m3j7vvhqeeky7wau/spack-src/_configtest.c:6: undefined referenc\n             e to `exp'\n>> 120     collect2: error: ld returned 1 exit status\n     121     failure.\n     122     removing: _configtest.c _configtest.o\n     123     C compiler: /p/vast1/koparasy/BUILD/head/spack/lib/spack/env/gcc/g\n             cc -Wno-unused-result -Wsign-compare -DNDEBUG -g -fwrapv -O3 -Wall\n              -fPIC -fPIC\n     124",
    "y/core -Inumpy/core/src/npymath -Inumpy/core/src/multiarray -Inump\n             y/core/src/umath -Inumpy/core/src/npysort -I/p/vast1/koparasy/BUIL\n             D/spack_cluster_installer/quartz/py-petsc4py/spack_env_dir/linux-r\n             hel7-broadwell/gcc-9.3.1/python-3.9.9-3o76m4utiyrsvuxwx2ytcv7sd3g3\n             5si6/include/python3.9 -c'\n     173     gcc: _configtest.c\n>> 174     _configtest.c:1:10: fatal error: sys/endian.h: No such file or dir\n             ectory\n     175         1 | #include <sys/endian.h>\n     176           |          ^~~~~~~~~~~~~~\n     177     compilation terminated.\n  >> 178     _configtest.c:1:10: fatal error: sys/endian.h: No such file or dir\n             ectory",
    "173     gcc: _configtest.c\n  >> 174     _configtest.c:1:10: fatal error: sys/endian.h: No such file or dir\n             ectory\n     175         1 | #include <sys/endian.h>\n     176           |          ^~~~~~~~~~~~~~\n     177     compilation terminated.\n>> 178     _configtest.c:1:10: fatal error: sys/endian.h: No such file or dir\n             ectory\n     179         1 | #include <sys/endian.h>\n     180           |          ^~~~~~~~~~~~~~\n     181     compilation terminated.\n     182     failure.\n     183     removing: _configtest.c _configtest.o",
    "y/core/src/umath -Inumpy/core/src/npysort -I/p/vast1/koparasy/BUIL\n             D/spack_cluster_installer/quartz/py-petsc4py/spack_env_dir/linux-r\n             hel7-broadwell/gcc-9.3.1/python-3.9.9-3o76m4utiyrsvuxwx2ytcv7sd3g3\n             5si6/include/python3.9 -c'\n     289     gcc: _configtest.c\n     290     _configtest.c: In function 'main':\n>> 291     _configtest.c:7:12: error: 'SIZEOF_LONGDOUBLE' undeclared (first u\n             se in this function); did you mean 'SIZEOF_LONG_DOUBLE'?\n     292         7 |     (void) SIZEOF_LONGDOUBLE;\n     293           |            ^~~~~~~~~~~~~~~~~\n     294           |            SIZEOF_LONG_DOUBLE\n     295     _configtest.c:7:12: note: each undeclared identifier is reported o\n             nly once for each function it appears in",
    "292         7 |     (void) SIZEOF_LONGDOUBLE;\n     293           |            ^~~~~~~~~~~~~~~~~\n     294           |            SIZEOF_LONG_DOUBLE\n     295     _configtest.c:7:12: note: each undeclared identifier is reported o\n             nly once for each function it appears in\n     296     _configtest.c: In function 'main':\n>> 297     _configtest.c:7:12: error: 'SIZEOF_LONGDOUBLE' undeclared (first u\n             se in this function); did you mean 'SIZEOF_LONG_DOUBLE'?\n     298         7 |     (void) SIZEOF_LONGDOUBLE;\n     299           |            ^~~~~~~~~~~~~~~~~\n     300           |            SIZEOF_LONG_DOUBLE\n     301     _configtest.c:7:12: note: each undeclared identifier is reported o\n             nly once for each function it appears in",
    "st.o -o _configtest\n     435     /usr/tce/packages/gcc/gcc-9.3.1/rh/usr/bin/../libexec/gcc/x86_64-r\n             edhat-linux/9/ld: _configtest.o: in function `main':\n  >> 436     /tmp/koparasy/spack-stage/spack-stage-py-numpy-1.14.1-2d2rt5cmgzkd\n             7ib5m3j7vvhqeeky7wau/spack-src/_configtest.c:6: undefined referenc\n             e to `exp'\n>> 437     collect2: error: ld returned 1 exit status\n     438     /usr/tce/packages/gcc/gcc-9.3.1/rh/usr/bin/../libexec/gcc/x86_64-r\n             edhat-linux/9/ld: _configtest.o: in function `main':\n  >> 439     /tmp/koparasy/spack-stage/spack-stage-py-numpy-1.14.1-2d2rt5cmgzkd\n             7ib5m3j7vvhqeeky7wau/spack-src/_configtest.c:6: undefined referenc\n             e to `exp'\n  >> 440     collect2: error: ld returned 1 exit status",
    ">> 437     collect2: error: ld returned 1 exit status\n     438     /usr/tce/packages/gcc/gcc-9.3.1/rh/usr/bin/../libexec/gcc/x86_64-r\n             edhat-linux/9/ld: _configtest.o: in function `main':\n  >> 439     /tmp/koparasy/spack-stage/spack-stage-py-numpy-1.14.1-2d2rt5cmgzkd\n             7ib5m3j7vvhqeeky7wau/spack-src/_configtest.c:6: undefined referenc\n             e to `exp'\n>> 440     collect2: error: ld returned 1 exit status\n     441     failure.\n     442     removing: _configtest.c _configtest.o\n     443     C compiler: /p/vast1/koparasy/BUILD/head/spack/lib/spack/env/gcc/g\n             cc -Wno-unused-result -Wsign-compare -DNDEBUG -g -fwrapv -O3 -Wall\n              -fPIC -fPIC\n     444",
    "869     gcc: _configtest.c\n     870     _configtest.c:3: warning: ignoring #pragma clang diagnostic [-Wunk\n             nown-pragmas]\n     871         3 | #pragma clang diagnostic error \"-Wattributes\"\n     872           |\n     873     _configtest.c: In function '__declspec':\n>> 874     _configtest.c:5:24: error: expected declaration specifiers before\n             'foo'\n     875         5 | int __declspec(thread) foo;\n     876           |                        ^~~\n  >> 877     _configtest.c:9:1: error: expected '=', ',', ';', 'asm' or '__attr\n             ibute__' before '{' token\n     878         9 | {",
    "872           |\n     873     _configtest.c: In function '__declspec':\n  >> 874     _configtest.c:5:24: error: expected declaration specifiers before\n             'foo'\n     875         5 | int __declspec(thread) foo;\n     876           |                        ^~~\n>> 877     _configtest.c:9:1: error: expected '=', ',', ';', 'asm' or '__attr\n             ibute__' before '{' token\n     878         9 | {\n     879           | ^\n     880     _configtest.c:5:5: warning: type of 'thread' defaults to 'int' [-W\n             implicit-int]\n     881         5 | int __declspec(thread) foo;",
    "878         9 | {\n     879           | ^\n     880     _configtest.c:5:5: warning: type of 'thread' defaults to 'int' [-W\n             implicit-int]\n     881         5 | int __declspec(thread) foo;\n     882           |     ^~~~~~~~~~\n>> 883     _configtest.c:11: error: expected '{' at end of input\n     884        11 | }\n     885           |\n     886     _configtest.c:11: warning: control reaches end of non-void functio\n             n [-Wreturn-type]\n     887        11 | }\n     888           |",
    "888           |\n     889     _configtest.c:3: warning: ignoring #pragma clang diagnostic [-Wunk\n             nown-pragmas]\n     890         3 | #pragma clang diagnostic error \"-Wattributes\"\n     891           |\n     892     _configtest.c: In function '__declspec':\n>> 893     _configtest.c:5:24: error: expected declaration specifiers before\n             'foo'\n     894         5 | int __declspec(thread) foo;\n     895           |                        ^~~\n  >> 896     _configtest.c:9:1: error: expected '=', ',', ';', 'asm' or '__attr\n             ibute__' before '{' token\n     897         9 | {",
    "891           |\n     892     _configtest.c: In function '__declspec':\n  >> 893     _configtest.c:5:24: error: expected declaration specifiers before\n             'foo'\n     894         5 | int __declspec(thread) foo;\n     895           |                        ^~~\n>> 896     _configtest.c:9:1: error: expected '=', ',', ';', 'asm' or '__attr\n             ibute__' before '{' token\n     897         9 | {\n     898           | ^\n     899     _configtest.c:5:5: warning: type of 'thread' defaults to 'int' [-W\n             implicit-int]\n     900         5 | int __declspec(thread) foo;",
    "897         9 | {\n     898           | ^\n     899     _configtest.c:5:5: warning: type of 'thread' defaults to 'int' [-W\n             implicit-int]\n     900         5 | int __declspec(thread) foo;\n     901           |     ^~~~~~~~~~\n>> 902     _configtest.c:11: error: expected '{' at end of input\n     903        11 | }\n     904           |\n     905     _configtest.c:11: warning: control reaches end of non-void functio\n             n [-Wreturn-type]\n     906        11 | }\n     907           |",
    "y/core/src/umath -Inumpy/core/src/npysort -I/p/vast1/koparasy/BUIL\n             D/spack_cluster_installer/quartz/py-petsc4py/spack_env_dir/linux-r\n             hel7-broadwell/gcc-9.3.1/python-3.9.9-3o76m4utiyrsvuxwx2ytcv7sd3g3\n             5si6/include/python3.9 -c'\n     1235    gcc: _configtest.c\n     1236    _configtest.c: In function 'main':\n>> 1237    _configtest.c:8:12: error: 'HAVE_DECL_SIGNBIT' undeclared (first u\n             se in this function); did you mean 'HAVE_DECL_ISNAN'?\n     1238        8 |     (void) HAVE_DECL_SIGNBIT;\n     1239          |            ^~~~~~~~~~~~~~~~~\n     1240          |            HAVE_DECL_ISNAN\n     1241    _configtest.c:8:12: note: each undeclared identifier is reported o\n             nly once for each function it appears in",
    "1238        8 |     (void) HAVE_DECL_SIGNBIT;\n     1239          |            ^~~~~~~~~~~~~~~~~\n     1240          |            HAVE_DECL_ISNAN\n     1241    _configtest.c:8:12: note: each undeclared identifier is reported o\n             nly once for each function it appears in\n     1242    _configtest.c: In function 'main':\n>> 1243    _configtest.c:8:12: error: 'HAVE_DECL_SIGNBIT' undeclared (first u\n             se in this function); did you mean 'HAVE_DECL_ISNAN'?\n     1244        8 |     (void) HAVE_DECL_SIGNBIT;\n     1245          |            ^~~~~~~~~~~~~~~~~\n     1246          |            HAVE_DECL_ISNAN\n     1247    _configtest.c:8:12: note: each undeclared identifier is reported o\n             nly once for each function it appears in",
    "st.o -o _configtest\n     1829    /usr/tce/packages/gcc/gcc-9.3.1/rh/usr/bin/../libexec/gcc/x86_64-r\n             edhat-linux/9/ld: _configtest.o: in function `main':\n  >> 1830    /tmp/koparasy/spack-stage/spack-stage-py-numpy-1.14.1-2d2rt5cmgzkd\n             7ib5m3j7vvhqeeky7wau/spack-src/_configtest.c:6: undefined referenc\n             e to `exp'\n>> 1831    collect2: error: ld returned 1 exit status\n     1832    /usr/tce/packages/gcc/gcc-9.3.1/rh/usr/bin/../libexec/gcc/x86_64-r\n             edhat-linux/9/ld: _configtest.o: in function `main':\n  >> 1833    /tmp/koparasy/spack-stage/spack-stage-py-numpy-1.14.1-2d2rt5cmgzkd\n             7ib5m3j7vvhqeeky7wau/spack-src/_configtest.c:6: undefined referenc\n             e to `exp'\n  >> 1834    collect2: error: ld returned 1 exit status",
    ">> 1831    collect2: error: ld returned 1 exit status\n     1832    /usr/tce/packages/gcc/gcc-9.3.1/rh/usr/bin/../libexec/gcc/x86_64-r\n             edhat-linux/9/ld: _configtest.o: in function `main':\n  >> 1833    /tmp/koparasy/spack-stage/spack-stage-py-numpy-1.14.1-2d2rt5cmgzkd\n             7ib5m3j7vvhqeeky7wau/spack-src/_configtest.c:6: undefined referenc\n             e to `exp'\n>> 1834    collect2: error: ld returned 1 exit status\n     1835    failure.\n     1836    removing: _configtest.c _configtest.o\n     1837    C compiler: /p/vast1/koparasy/BUILD/head/spack/lib/spack/env/gcc/g\n             cc -Wno-unused-result -Wsign-compare -DNDEBUG -g -fwrapv -O3 -Wall\n              -fPIC -fPIC\n     1838",
    "pymath -Ibuild/src.linux-x86_64-3.9/numpy/core/src/private -Ibuild\n             /src.linux-x86_64-3.9/numpy/core/src/npymath -Ibuild/src.linux-x86\n             _64-3.9/numpy/core/src/private -Ibuild/src.linux-x86_64-3.9/numpy/\n             core/src/npymath -c'\n     4423    gcc: numpy/random/mtrand/mtrand.c\n     4424    numpy/random/mtrand/mtrand.c: In function 'PyInit_mtrand':\n>> 4425    numpy/random/mtrand/mtrand.c:43644:33: error: 'PyTypeObject' {aka\n             'struct _typeobject'} has no member named 'tp_print'\n     4426    43644 |   __pyx_type_6mtrand_RandomState.tp_print = 0;\n     4427          |                                 ^\n     4428    numpy/random/mtrand/mtrand.c: In function '__Pyx_ParseOptionalKeyw\n             ords':\n     4429    numpy/random/mtrand/mtrand.c:45175:21: warning: '_PyUnicode_get_ws",
    "utiyrsvuxwx2ytcv7sd3g35si6/include/python3.9/cpython/unicodeobject\n             .h:446:26: note: declared here\n     4535      446 | static inline Py_ssize_t _PyUnicode_get_wstr_length(PyObje\n             ct *op) {\n     4536          |                          ^~~~~~~~~~~~~~~~~~~~~~~~~~\n     4537    numpy/random/mtrand/mtrand.c: In function '__Pyx__ExceptionSave':\n>> 4538    numpy/random/mtrand/mtrand.c:45472:21: error: 'PyThreadState' {aka\n              'struct _ts'} has no member named 'exc_type'; did you mean 'curex\n             c_type'?\n     4539    45472 |     *type = tstate->exc_type;\n     4540          |                     ^~~~~~~~\n     4541          |                     curexc_type\n  >> 4542    numpy/random/mtrand/mtrand.c:45473:22: error: 'PyThreadState' {aka",
    ">> 4538    numpy/random/mtrand/mtrand.c:45472:21: error: 'PyThreadState' {aka\n              'struct _ts'} has no member named 'exc_type'; did you mean 'curex\n             c_type'?\n     4539    45472 |     *type = tstate->exc_type;\n     4540          |                     ^~~~~~~~\n     4541          |                     curexc_type\n>> 4542    numpy/random/mtrand/mtrand.c:45473:22: error: 'PyThreadState' {aka\n              'struct _ts'} has no member named 'exc_value'; did you mean 'cure\n             xc_value'?\n     4543    45473 |     *value = tstate->exc_value;\n     4544          |                      ^~~~~~~~~\n     4545          |                      curexc_value\n  >> 4546    numpy/random/mtrand/mtrand.c:45474:19: error: 'PyThreadState' {aka",
    ">> 4542    numpy/random/mtrand/mtrand.c:45473:22: error: 'PyThreadState' {aka\n              'struct _ts'} has no member named 'exc_value'; did you mean 'cure\n             xc_value'?\n     4543    45473 |     *value = tstate->exc_value;\n     4544          |                      ^~~~~~~~~\n     4545          |                      curexc_value\n>> 4546    numpy/random/mtrand/mtrand.c:45474:19: error: 'PyThreadState' {aka\n              'struct _ts'} has no member named 'exc_traceback'; did you mean '\n             curexc_traceback'?\n     4547    45474 |     *tb = tstate->exc_traceback;\n     4548          |                   ^~~~~~~~~~~~~\n     4549          |                   curexc_traceback\n     4550    numpy/random/mtrand/mtrand.c: In function '__Pyx__ExceptionReset':",
    "'struct _ts'} has no member named 'exc_traceback'; did you mean '\n             curexc_traceback'?\n     4547    45474 |     *tb = tstate->exc_traceback;\n     4548          |                   ^~~~~~~~~~~~~\n     4549          |                   curexc_traceback\n     4550    numpy/random/mtrand/mtrand.c: In function '__Pyx__ExceptionReset':\n>> 4551    numpy/random/mtrand/mtrand.c:45481:24: error: 'PyThreadState' {aka\n              'struct _ts'} has no member named 'exc_type'; did you mean 'curex\n             c_type'?\n     4552    45481 |     tmp_type = tstate->exc_type;\n     4553          |                        ^~~~~~~~\n     4554          |                        curexc_type\n  >> 4555    numpy/random/mtrand/mtrand.c:45482:25: error: 'PyThreadState' {aka",
    ">> 4551    numpy/random/mtrand/mtrand.c:45481:24: error: 'PyThreadState' {aka\n              'struct _ts'} has no member named 'exc_type'; did you mean 'curex\n             c_type'?\n     4552    45481 |     tmp_type = tstate->exc_type;\n     4553          |                        ^~~~~~~~\n     4554          |                        curexc_type\n>> 4555    numpy/random/mtrand/mtrand.c:45482:25: error: 'PyThreadState' {aka\n              'struct _ts'} has no member named 'exc_value'; did you mean 'cure\n             xc_value'?\n     4556    45482 |     tmp_value = tstate->exc_value;\n     4557          |                         ^~~~~~~~~\n     4558          |                         curexc_value\n  >> 4559    numpy/random/mtrand/mtrand.c:45483:22: error: 'PyThreadState' {aka",
    ">> 4555    numpy/random/mtrand/mtrand.c:45482:25: error: 'PyThreadState' {aka\n              'struct _ts'} has no member named 'exc_value'; did you mean 'cure\n             xc_value'?\n     4556    45482 |     tmp_value = tstate->exc_value;\n     4557          |                         ^~~~~~~~~\n     4558          |                         curexc_value\n>> 4559    numpy/random/mtrand/mtrand.c:45483:22: error: 'PyThreadState' {aka\n              'struct _ts'} has no member named 'exc_traceback'; did you mean '\n             curexc_traceback'?\n     4560    45483 |     tmp_tb = tstate->exc_traceback;\n     4561          |                      ^~~~~~~~~~~~~\n     4562          |                      curexc_traceback\n  >> 4563    numpy/random/mtrand/mtrand.c:45484:13: error: 'PyThreadState' {aka",
    ">> 4559    numpy/random/mtrand/mtrand.c:45483:22: error: 'PyThreadState' {aka\n              'struct _ts'} has no member named 'exc_traceback'; did you mean '\n             curexc_traceback'?\n     4560    45483 |     tmp_tb = tstate->exc_traceback;\n     4561          |                      ^~~~~~~~~~~~~\n     4562          |                      curexc_traceback\n>> 4563    numpy/random/mtrand/mtrand.c:45484:13: error: 'PyThreadState' {aka\n              'struct _ts'} has no member named 'exc_type'; did you mean 'curex\n             c_type'?\n     4564    45484 |     tstate->exc_type = type;\n     4565          |             ^~~~~~~~\n     4566          |             curexc_type\n  >> 4567    numpy/random/mtrand/mtrand.c:45485:13: error: 'PyThreadState' {aka",
    ">> 4563    numpy/random/mtrand/mtrand.c:45484:13: error: 'PyThreadState' {aka\n              'struct _ts'} has no member named 'exc_type'; did you mean 'curex\n             c_type'?\n     4564    45484 |     tstate->exc_type = type;\n     4565          |             ^~~~~~~~\n     4566          |             curexc_type\n>> 4567    numpy/random/mtrand/mtrand.c:45485:13: error: 'PyThreadState' {aka\n              'struct _ts'} has no member named 'exc_value'; did you mean 'cure\n             xc_value'?\n     4568    45485 |     tstate->exc_value = value;\n     4569          |             ^~~~~~~~~\n     4570          |             curexc_value\n  >> 4571    numpy/random/mtrand/mtrand.c:45486:13: error: 'PyThreadState' {aka",
    ">> 4567    numpy/random/mtrand/mtrand.c:45485:13: error: 'PyThreadState' {aka\n              'struct _ts'} has no member named 'exc_value'; did you mean 'cure\n             xc_value'?\n     4568    45485 |     tstate->exc_value = value;\n     4569          |             ^~~~~~~~~\n     4570          |             curexc_value\n>> 4571    numpy/random/mtrand/mtrand.c:45486:13: error: 'PyThreadState' {aka\n              'struct _ts'} has no member named 'exc_traceback'; did you mean '\n             curexc_traceback'?\n     4572    45486 |     tstate->exc_traceback = tb;\n     4573          |             ^~~~~~~~~~~~~\n     4574          |             curexc_traceback\n     4575    numpy/random/mtrand/mtrand.c: In function '__Pyx__GetException':",
    "'struct _ts'} has no member named 'exc_traceback'; did you mean '\n             curexc_traceback'?\n     4572    45486 |     tstate->exc_traceback = tb;\n     4573          |             ^~~~~~~~~~~~~\n     4574          |             curexc_traceback\n     4575    numpy/random/mtrand/mtrand.c: In function '__Pyx__GetException':\n>> 4576    numpy/random/mtrand/mtrand.c:45541:24: error: 'PyThreadState' {aka\n              'struct _ts'} has no member named 'exc_type'; did you mean 'curex\n             c_type'?\n     4577    45541 |     tmp_type = tstate->exc_type;\n     4578          |                        ^~~~~~~~\n     4579          |                        curexc_type\n  >> 4580    numpy/random/mtrand/mtrand.c:45542:25: error: 'PyThreadState' {aka",
    ">> 4576    numpy/random/mtrand/mtrand.c:45541:24: error: 'PyThreadState' {aka\n              'struct _ts'} has no member named 'exc_type'; did you mean 'curex\n             c_type'?\n     4577    45541 |     tmp_type = tstate->exc_type;\n     4578          |                        ^~~~~~~~\n     4579          |                        curexc_type\n>> 4580    numpy/random/mtrand/mtrand.c:45542:25: error: 'PyThreadState' {aka\n              'struct _ts'} has no member named 'exc_value'; did you mean 'cure\n             xc_value'?\n     4581    45542 |     tmp_value = tstate->exc_value;\n     4582          |                         ^~~~~~~~~\n     4583          |                         curexc_value\n  >> 4584    numpy/random/mtrand/mtrand.c:45543:22: error: 'PyThreadState' {aka",
    ">> 4580    numpy/random/mtrand/mtrand.c:45542:25: error: 'PyThreadState' {aka\n              'struct _ts'} has no member named 'exc_value'; did you mean 'cure\n             xc_value'?\n     4581    45542 |     tmp_value = tstate->exc_value;\n     4582          |                         ^~~~~~~~~\n     4583          |                         curexc_value\n>> 4584    numpy/random/mtrand/mtrand.c:45543:22: error: 'PyThreadState' {aka\n              'struct _ts'} has no member named 'exc_traceback'; did you mean '\n             curexc_traceback'?\n     4585    45543 |     tmp_tb = tstate->exc_traceback;\n     4586          |                      ^~~~~~~~~~~~~\n     4587          |                      curexc_traceback\n  >> 4588    numpy/random/mtrand/mtrand.c:45544:13: error: 'PyThreadState' {aka",
    ">> 4584    numpy/random/mtrand/mtrand.c:45543:22: error: 'PyThreadState' {aka\n              'struct _ts'} has no member named 'exc_traceback'; did you mean '\n             curexc_traceback'?\n     4585    45543 |     tmp_tb = tstate->exc_traceback;\n     4586          |                      ^~~~~~~~~~~~~\n     4587          |                      curexc_traceback\n>> 4588    numpy/random/mtrand/mtrand.c:45544:13: error: 'PyThreadState' {aka\n              'struct _ts'} has no member named 'exc_type'; did you mean 'curex\n             c_type'?\n     4589    45544 |     tstate->exc_type = local_type;\n     4590          |             ^~~~~~~~\n     4591          |             curexc_type\n  >> 4592    numpy/random/mtrand/mtrand.c:45545:13: error: 'PyThreadState' {aka",
    ">> 4588    numpy/random/mtrand/mtrand.c:45544:13: error: 'PyThreadState' {aka\n              'struct _ts'} has no member named 'exc_type'; did you mean 'curex\n             c_type'?\n     4589    45544 |     tstate->exc_type = local_type;\n     4590          |             ^~~~~~~~\n     4591          |             curexc_type\n>> 4592    numpy/random/mtrand/mtrand.c:45545:13: error: 'PyThreadState' {aka\n              'struct _ts'} has no member named 'exc_value'; did you mean 'cure\n             xc_value'?\n     4593    45545 |     tstate->exc_value = local_value;\n     4594          |             ^~~~~~~~~\n     4595          |             curexc_value\n  >> 4596    numpy/random/mtrand/mtrand.c:45546:13: error: 'PyThreadState' {aka",
    ">> 4592    numpy/random/mtrand/mtrand.c:45545:13: error: 'PyThreadState' {aka\n              'struct _ts'} has no member named 'exc_value'; did you mean 'cure\n             xc_value'?\n     4593    45545 |     tstate->exc_value = local_value;\n     4594          |             ^~~~~~~~~\n     4595          |             curexc_value\n>> 4596    numpy/random/mtrand/mtrand.c:45546:13: error: 'PyThreadState' {aka\n              'struct _ts'} has no member named 'exc_traceback'; did you mean '\n             curexc_traceback'?\n     4597    45546 |     tstate->exc_traceback = local_tb;\n     4598          |             ^~~~~~~~~~~~~\n     4599          |             curexc_traceback\n     4600    numpy/random/mtrand/mtrand.c: In function 'PyInit_mtrand':",
    "'struct _ts'} has no member named 'exc_traceback'; did you mean '\n             curexc_traceback'?\n     4597    45546 |     tstate->exc_traceback = local_tb;\n     4598          |             ^~~~~~~~~~~~~\n     4599          |             curexc_traceback\n     4600    numpy/random/mtrand/mtrand.c: In function 'PyInit_mtrand':\n>> 4601    numpy/random/mtrand/mtrand.c:43644:33: error: 'PyTypeObject' {aka\n             'struct _typeobject'} has no member named 'tp_print'\n     4602    43644 |   __pyx_type_6mtrand_RandomState.tp_print = 0;\n     4603          |                                 ^\n     4604    numpy/random/mtrand/mtrand.c: In function '__Pyx_ParseOptionalKeyw\n             ords':\n     4605    numpy/random/mtrand/mtrand.c:45175:21: warning: '_PyUnicode_get_ws",
    "utiyrsvuxwx2ytcv7sd3g35si6/include/python3.9/cpython/unicodeobject\n             .h:446:26: note: declared here\n     4711      446 | static inline Py_ssize_t _PyUnicode_get_wstr_length(PyObje\n             ct *op) {\n     4712          |                          ^~~~~~~~~~~~~~~~~~~~~~~~~~\n     4713    numpy/random/mtrand/mtrand.c: In function '__Pyx__ExceptionSave':\n>> 4714    numpy/random/mtrand/mtrand.c:45472:21: error: 'PyThreadState' {aka\n              'struct _ts'} has no member named 'exc_type'; did you mean 'curex\n             c_type'?\n     4715    45472 |     *type = tstate->exc_type;\n     4716          |                     ^~~~~~~~\n     4717          |                     curexc_type\n  >> 4718    numpy/random/mtrand/mtrand.c:45473:22: error: 'PyThreadState' {aka",
    ">> 4714    numpy/random/mtrand/mtrand.c:45472:21: error: 'PyThreadState' {aka\n              'struct _ts'} has no member named 'exc_type'; did you mean 'curex\n             c_type'?\n     4715    45472 |     *type = tstate->exc_type;\n     4716          |                     ^~~~~~~~\n     4717          |                     curexc_type\n>> 4718    numpy/random/mtrand/mtrand.c:45473:22: error: 'PyThreadState' {aka\n              'struct _ts'} has no member named 'exc_value'; did you mean 'cure\n             xc_value'?\n     4719    45473 |     *value = tstate->exc_value;\n     4720          |                      ^~~~~~~~~\n     4721          |                      curexc_value\n  >> 4722    numpy/random/mtrand/mtrand.c:45474:19: error: 'PyThreadState' {aka",
    ">> 4718    numpy/random/mtrand/mtrand.c:45473:22: error: 'PyThreadState' {aka\n              'struct _ts'} has no member named 'exc_value'; did you mean 'cure\n             xc_value'?\n     4719    45473 |     *value = tstate->exc_value;\n     4720          |                      ^~~~~~~~~\n     4721          |                      curexc_value\n>> 4722    numpy/random/mtrand/mtrand.c:45474:19: error: 'PyThreadState' {aka\n              'struct _ts'} has no member named 'exc_traceback'; did you mean '\n             curexc_traceback'?\n     4723    45474 |     *tb = tstate->exc_traceback;\n     4724          |                   ^~~~~~~~~~~~~\n     4725          |                   curexc_traceback\n     4726    numpy/random/mtrand/mtrand.c: In function '__Pyx__ExceptionReset':",
    "'struct _ts'} has no member named 'exc_traceback'; did you mean '\n             curexc_traceback'?\n     4723    45474 |     *tb = tstate->exc_traceback;\n     4724          |                   ^~~~~~~~~~~~~\n     4725          |                   curexc_traceback\n     4726    numpy/random/mtrand/mtrand.c: In function '__Pyx__ExceptionReset':\n>> 4727    numpy/random/mtrand/mtrand.c:45481:24: error: 'PyThreadState' {aka\n              'struct _ts'} has no member named 'exc_type'; did you mean 'curex\n             c_type'?\n     4728    45481 |     tmp_type = tstate->exc_type;\n     4729          |                        ^~~~~~~~\n     4730          |                        curexc_type\n  >> 4731    numpy/random/mtrand/mtrand.c:45482:25: error: 'PyThreadState' {aka",
    ">> 4727    numpy/random/mtrand/mtrand.c:45481:24: error: 'PyThreadState' {aka\n              'struct _ts'} has no member named 'exc_type'; did you mean 'curex\n             c_type'?\n     4728    45481 |     tmp_type = tstate->exc_type;\n     4729          |                        ^~~~~~~~\n     4730          |                        curexc_type\n>> 4731    numpy/random/mtrand/mtrand.c:45482:25: error: 'PyThreadState' {aka\n              'struct _ts'} has no member named 'exc_value'; did you mean 'cure\n             xc_value'?\n     4732    45482 |     tmp_value = tstate->exc_value;\n     4733          |                         ^~~~~~~~~\n     4734          |                         curexc_value\n  >> 4735    numpy/random/mtrand/mtrand.c:45483:22: error: 'PyThreadState' {aka",
    ">> 4731    numpy/random/mtrand/mtrand.c:45482:25: error: 'PyThreadState' {aka\n              'struct _ts'} has no member named 'exc_value'; did you mean 'cure\n             xc_value'?\n     4732    45482 |     tmp_value = tstate->exc_value;\n     4733          |                         ^~~~~~~~~\n     4734          |                         curexc_value\n>> 4735    numpy/random/mtrand/mtrand.c:45483:22: error: 'PyThreadState' {aka\n              'struct _ts'} has no member named 'exc_traceback'; did you mean '\n             curexc_traceback'?\n     4736    45483 |     tmp_tb = tstate->exc_traceback;\n     4737          |                      ^~~~~~~~~~~~~\n     4738          |                      curexc_traceback\n  >> 4739    numpy/random/mtrand/mtrand.c:45484:13: error: 'PyThreadState' {aka",
    ">> 4735    numpy/random/mtrand/mtrand.c:45483:22: error: 'PyThreadState' {aka\n              'struct _ts'} has no member named 'exc_traceback'; did you mean '\n             curexc_traceback'?\n     4736    45483 |     tmp_tb = tstate->exc_traceback;\n     4737          |                      ^~~~~~~~~~~~~\n     4738          |                      curexc_traceback\n>> 4739    numpy/random/mtrand/mtrand.c:45484:13: error: 'PyThreadState' {aka\n              'struct _ts'} has no member named 'exc_type'; did you mean 'curex\n             c_type'?\n     4740    45484 |     tstate->exc_type = type;\n     4741          |             ^~~~~~~~\n     4742          |             curexc_type\n  >> 4743    numpy/random/mtrand/mtrand.c:45485:13: error: 'PyThreadState' {aka",
    ">> 4739    numpy/random/mtrand/mtrand.c:45484:13: error: 'PyThreadState' {aka\n              'struct _ts'} has no member named 'exc_type'; did you mean 'curex\n             c_type'?\n     4740    45484 |     tstate->exc_type = type;\n     4741          |             ^~~~~~~~\n     4742          |             curexc_type\n>> 4743    numpy/random/mtrand/mtrand.c:45485:13: error: 'PyThreadState' {aka\n              'struct _ts'} has no member named 'exc_value'; did you mean 'cure\n             xc_value'?\n     4744    45485 |     tstate->exc_value = value;\n     4745          |             ^~~~~~~~~\n     4746          |             curexc_value\n  >> 4747    numpy/random/mtrand/mtrand.c:45486:13: error: 'PyThreadState' {aka",
    ">> 4743    numpy/random/mtrand/mtrand.c:45485:13: error: 'PyThreadState' {aka\n              'struct _ts'} has no member named 'exc_value'; did you mean 'cure\n             xc_value'?\n     4744    45485 |     tstate->exc_value = value;\n     4745          |             ^~~~~~~~~\n     4746          |             curexc_value\n>> 4747    numpy/random/mtrand/mtrand.c:45486:13: error: 'PyThreadState' {aka\n              'struct _ts'} has no member named 'exc_traceback'; did you mean '\n             curexc_traceback'?\n     4748    45486 |     tstate->exc_traceback = tb;\n     4749          |             ^~~~~~~~~~~~~\n     4750          |             curexc_traceback\n     4751    numpy/random/mtrand/mtrand.c: In function '__Pyx__GetException':",
    "'struct _ts'} has no member named 'exc_traceback'; did you mean '\n             curexc_traceback'?\n     4748    45486 |     tstate->exc_traceback = tb;\n     4749          |             ^~~~~~~~~~~~~\n     4750          |             curexc_traceback\n     4751    numpy/random/mtrand/mtrand.c: In function '__Pyx__GetException':\n>> 4752    numpy/random/mtrand/mtrand.c:45541:24: error: 'PyThreadState' {aka\n              'struct _ts'} has no member named 'exc_type'; did you mean 'curex\n             c_type'?\n     4753    45541 |     tmp_type = tstate->exc_type;\n     4754          |                        ^~~~~~~~\n     4755          |                        curexc_type\n  >> 4756    numpy/random/mtrand/mtrand.c:45542:25: error: 'PyThreadState' {aka",
    ">> 4752    numpy/random/mtrand/mtrand.c:45541:24: error: 'PyThreadState' {aka\n              'struct _ts'} has no member named 'exc_type'; did you mean 'curex\n             c_type'?\n     4753    45541 |     tmp_type = tstate->exc_type;\n     4754          |                        ^~~~~~~~\n     4755          |                        curexc_type\n>> 4756    numpy/random/mtrand/mtrand.c:45542:25: error: 'PyThreadState' {aka\n              'struct _ts'} has no member named 'exc_value'; did you mean 'cure\n             xc_value'?\n     4757    45542 |     tmp_value = tstate->exc_value;\n     4758          |                         ^~~~~~~~~\n     4759          |                         curexc_value\n  >> 4760    numpy/random/mtrand/mtrand.c:45543:22: error: 'PyThreadState' {aka",
    ">> 4756    numpy/random/mtrand/mtrand.c:45542:25: error: 'PyThreadState' {aka\n              'struct _ts'} has no member named 'exc_value'; did you mean 'cure\n             xc_value'?\n     4757    45542 |     tmp_value = tstate->exc_value;\n     4758          |                         ^~~~~~~~~\n     4759          |                         curexc_value\n>> 4760    numpy/random/mtrand/mtrand.c:45543:22: error: 'PyThreadState' {aka\n              'struct _ts'} has no member named 'exc_traceback'; did you mean '\n             curexc_traceback'?\n     4761    45543 |     tmp_tb = tstate->exc_traceback;\n     4762          |                      ^~~~~~~~~~~~~\n     4763          |                      curexc_traceback\n  >> 4764    numpy/random/mtrand/mtrand.c:45544:13: error: 'PyThreadState' {aka",
    ">> 4760    numpy/random/mtrand/mtrand.c:45543:22: error: 'PyThreadState' {aka\n              'struct _ts'} has no member named 'exc_traceback'; did you mean '\n             curexc_traceback'?\n     4761    45543 |     tmp_tb = tstate->exc_traceback;\n     4762          |                      ^~~~~~~~~~~~~\n     4763          |                      curexc_traceback\n>> 4764    numpy/random/mtrand/mtrand.c:45544:13: error: 'PyThreadState' {aka\n              'struct _ts'} has no member named 'exc_type'; did you mean 'curex\n             c_type'?\n     4765    45544 |     tstate->exc_type = local_type;\n     4766          |             ^~~~~~~~\n     4767          |             curexc_type\n  >> 4768    numpy/random/mtrand/mtrand.c:45545:13: error: 'PyThreadState' {aka",
    ">> 4764    numpy/random/mtrand/mtrand.c:45544:13: error: 'PyThreadState' {aka\n              'struct _ts'} has no member named 'exc_type'; did you mean 'curex\n             c_type'?\n     4765    45544 |     tstate->exc_type = local_type;\n     4766          |             ^~~~~~~~\n     4767          |             curexc_type\n>> 4768    numpy/random/mtrand/mtrand.c:45545:13: error: 'PyThreadState' {aka\n              'struct _ts'} has no member named 'exc_value'; did you mean 'cure\n             xc_value'?\n     4769    45545 |     tstate->exc_value = local_value;\n     4770          |             ^~~~~~~~~\n     4771          |             curexc_value\n  >> 4772    numpy/random/mtrand/mtrand.c:45546:13: error: 'PyThreadState' {aka",
    ">> 4768    numpy/random/mtrand/mtrand.c:45545:13: error: 'PyThreadState' {aka\n              'struct _ts'} has no member named 'exc_value'; did you mean 'cure\n             xc_value'?\n     4769    45545 |     tstate->exc_value = local_value;\n     4770          |             ^~~~~~~~~\n     4771          |             curexc_value\n>> 4772    numpy/random/mtrand/mtrand.c:45546:13: error: 'PyThreadState' {aka\n              'struct _ts'} has no member named 'exc_traceback'; did you mean '\n             curexc_traceback'?\n     4773    45546 |     tstate->exc_traceback = local_tb;\n     4774          |             ^~~~~~~~~~~~~\n     4775          |             curexc_traceback\n  >> 4776    error: Command \"/p/vast1/koparasy/BUILD/head/spack/lib/spack/env/g",
    "st.o -o _configtest\n     115     /usr/tce/packages/gcc/gcc-9.3.1/rh/usr/bin/../libexec/gcc/x86_64-r\n             edhat-linux/9/ld: _configtest.o: in function `main':\n  >> 116     /tmp/koparasy/spack-stage/spack-stage-py-numpy-1.14.1-2d2rt5cmgzkd\n             7ib5m3j7vvhqeeky7wau/spack-src/_configtest.c:6: undefined referenc\n             e to `exp'\n>> 117     collect2: error: ld returned 1 exit status\n     118     /usr/tce/packages/gcc/gcc-9.3.1/rh/usr/bin/../libexec/gcc/x86_64-r\n             edhat-linux/9/ld: _configtest.o: in function `main':\n  >> 119     /tmp/koparasy/spack-stage/spack-stage-py-numpy-1.14.1-2d2rt5cmgzkd\n             7ib5m3j7vvhqeeky7wau/spack-src/_configtest.c:6: undefined referenc\n             e to `exp'\n  >> 120     collect2: error: ld returned 1 exit status",
    ">> 117     collect2: error: ld returned 1 exit status\n     118     /usr/tce/packages/gcc/gcc-9.3.1/rh/usr/bin/../libexec/gcc/x86_64-r\n             edhat-linux/9/ld: _configtest.o: in function `main':\n  >> 119     /tmp/koparasy/spack-stage/spack-stage-py-numpy-1.14.1-2d2rt5cmgzkd\n             7ib5m3j7vvhqeeky7wau/spack-src/_configtest.c:6: undefined referenc\n             e to `exp'\n>> 120     collect2: error: ld returned 1 exit status\n     121     failure.\n     122     removing: _configtest.c _configtest.o\n     123     C compiler: /p/vast1/koparasy/BUILD/head/spack/lib/spack/env/gcc/g\n             cc -Wno-unused-result -Wsign-compare -DNDEBUG -g -fwrapv -O3 -Wall\n              -fPIC -fPIC\n     124",
    "y/core -Inumpy/core/src/npymath -Inumpy/core/src/multiarray -Inump\n             y/core/src/umath -Inumpy/core/src/npysort -I/p/vast1/koparasy/BUIL\n             D/spack_cluster_installer/quartz/py-warpx/spack_env_dir/linux-rhel\n             7-broadwell/gcc-9.3.1/python-3.9.9-3o76m4utiyrsvuxwx2ytcv7sd3g35si\n             6/include/python3.9 -c'\n     173     gcc: _configtest.c\n>> 174     _configtest.c:1:10: fatal error: sys/endian.h: No such file or dir\n             ectory\n     175         1 | #include <sys/endian.h>\n     176           |          ^~~~~~~~~~~~~~\n     177     compilation terminated.\n  >> 178     _configtest.c:1:10: fatal error: sys/endian.h: No such file or dir\n             ectory",
    "173     gcc: _configtest.c\n  >> 174     _configtest.c:1:10: fatal error: sys/endian.h: No such file or dir\n             ectory\n     175         1 | #include <sys/endian.h>\n     176           |          ^~~~~~~~~~~~~~\n     177     compilation terminated.\n>> 178     _configtest.c:1:10: fatal error: sys/endian.h: No such file or dir\n             ectory\n     179         1 | #include <sys/endian.h>\n     180           |          ^~~~~~~~~~~~~~\n     181     compilation terminated.\n     182     failure.\n     183     removing: _configtest.c _configtest.o",
    "y/core/src/umath -Inumpy/core/src/npysort -I/p/vast1/koparasy/BUIL\n             D/spack_cluster_installer/quartz/py-warpx/spack_env_dir/linux-rhel\n             7-broadwell/gcc-9.3.1/python-3.9.9-3o76m4utiyrsvuxwx2ytcv7sd3g35si\n             6/include/python3.9 -c'\n     289     gcc: _configtest.c\n     290     _configtest.c: In function 'main':\n>> 291     _configtest.c:7:12: error: 'SIZEOF_LONGDOUBLE' undeclared (first u\n             se in this function); did you mean 'SIZEOF_LONG_DOUBLE'?\n     292         7 |     (void) SIZEOF_LONGDOUBLE;\n     293           |            ^~~~~~~~~~~~~~~~~\n     294           |            SIZEOF_LONG_DOUBLE\n     295     _configtest.c:7:12: note: each undeclared identifier is reported o\n             nly once for each function it appears in",
    "292         7 |     (void) SIZEOF_LONGDOUBLE;\n     293           |            ^~~~~~~~~~~~~~~~~\n     294           |            SIZEOF_LONG_DOUBLE\n     295     _configtest.c:7:12: note: each undeclared identifier is reported o\n             nly once for each function it appears in\n     296     _configtest.c: In function 'main':\n>> 297     _configtest.c:7:12: error: 'SIZEOF_LONGDOUBLE' undeclared (first u\n             se in this function); did you mean 'SIZEOF_LONG_DOUBLE'?\n     298         7 |     (void) SIZEOF_LONGDOUBLE;\n     299           |            ^~~~~~~~~~~~~~~~~\n     300           |            SIZEOF_LONG_DOUBLE\n     301     _configtest.c:7:12: note: each undeclared identifier is reported o\n             nly once for each function it appears in",
    "st.o -o _configtest\n     435     /usr/tce/packages/gcc/gcc-9.3.1/rh/usr/bin/../libexec/gcc/x86_64-r\n             edhat-linux/9/ld: _configtest.o: in function `main':\n  >> 436     /tmp/koparasy/spack-stage/spack-stage-py-numpy-1.14.1-2d2rt5cmgzkd\n             7ib5m3j7vvhqeeky7wau/spack-src/_configtest.c:6: undefined referenc\n             e to `exp'\n>> 437     collect2: error: ld returned 1 exit status\n     438     /usr/tce/packages/gcc/gcc-9.3.1/rh/usr/bin/../libexec/gcc/x86_64-r\n             edhat-linux/9/ld: _configtest.o: in function `main':\n  >> 439     /tmp/koparasy/spack-stage/spack-stage-py-numpy-1.14.1-2d2rt5cmgzkd\n             7ib5m3j7vvhqeeky7wau/spack-src/_configtest.c:6: undefined referenc\n             e to `exp'\n  >> 440     collect2: error: ld returned 1 exit status",
    ">> 437     collect2: error: ld returned 1 exit status\n     438     /usr/tce/packages/gcc/gcc-9.3.1/rh/usr/bin/../libexec/gcc/x86_64-r\n             edhat-linux/9/ld: _configtest.o: in function `main':\n  >> 439     /tmp/koparasy/spack-stage/spack-stage-py-numpy-1.14.1-2d2rt5cmgzkd\n             7ib5m3j7vvhqeeky7wau/spack-src/_configtest.c:6: undefined referenc\n             e to `exp'\n>> 440     collect2: error: ld returned 1 exit status\n     441     failure.\n     442     removing: _configtest.c _configtest.o\n     443     C compiler: /p/vast1/koparasy/BUILD/head/spack/lib/spack/env/gcc/g\n             cc -Wno-unused-result -Wsign-compare -DNDEBUG -g -fwrapv -O3 -Wall\n              -fPIC -fPIC\n     444",
    "869     gcc: _configtest.c\n     870     _configtest.c:3: warning: ignoring #pragma clang diagnostic [-Wunk\n             nown-pragmas]\n     871         3 | #pragma clang diagnostic error \"-Wattributes\"\n     872           |\n     873     _configtest.c: In function '__declspec':\n>> 874     _configtest.c:5:24: error: expected declaration specifiers before\n             'foo'\n     875         5 | int __declspec(thread) foo;\n     876           |                        ^~~\n  >> 877     _configtest.c:9:1: error: expected '=', ',', ';', 'asm' or '__attr\n             ibute__' before '{' token\n     878         9 | {",
    "872           |\n     873     _configtest.c: In function '__declspec':\n  >> 874     _configtest.c:5:24: error: expected declaration specifiers before\n             'foo'\n     875         5 | int __declspec(thread) foo;\n     876           |                        ^~~\n>> 877     _configtest.c:9:1: error: expected '=', ',', ';', 'asm' or '__attr\n             ibute__' before '{' token\n     878         9 | {\n     879           | ^\n     880     _configtest.c:5:5: warning: type of 'thread' defaults to 'int' [-W\n             implicit-int]\n     881         5 | int __declspec(thread) foo;",
    "878         9 | {\n     879           | ^\n     880     _configtest.c:5:5: warning: type of 'thread' defaults to 'int' [-W\n             implicit-int]\n     881         5 | int __declspec(thread) foo;\n     882           |     ^~~~~~~~~~\n>> 883     _configtest.c:11: error: expected '{' at end of input\n     884        11 | }\n     885           |\n     886     _configtest.c:11: warning: control reaches end of non-void functio\n             n [-Wreturn-type]\n     887        11 | }\n     888           |",
    "888           |\n     889     _configtest.c:3: warning: ignoring #pragma clang diagnostic [-Wunk\n             nown-pragmas]\n     890         3 | #pragma clang diagnostic error \"-Wattributes\"\n     891           |\n     892     _configtest.c: In function '__declspec':\n>> 893     _configtest.c:5:24: error: expected declaration specifiers before\n             'foo'\n     894         5 | int __declspec(thread) foo;\n     895           |                        ^~~\n  >> 896     _configtest.c:9:1: error: expected '=', ',', ';', 'asm' or '__attr\n             ibute__' before '{' token\n     897         9 | {",
    "891           |\n     892     _configtest.c: In function '__declspec':\n  >> 893     _configtest.c:5:24: error: expected declaration specifiers before\n             'foo'\n     894         5 | int __declspec(thread) foo;\n     895           |                        ^~~\n>> 896     _configtest.c:9:1: error: expected '=', ',', ';', 'asm' or '__attr\n             ibute__' before '{' token\n     897         9 | {\n     898           | ^\n     899     _configtest.c:5:5: warning: type of 'thread' defaults to 'int' [-W\n             implicit-int]\n     900         5 | int __declspec(thread) foo;",
    "897         9 | {\n     898           | ^\n     899     _configtest.c:5:5: warning: type of 'thread' defaults to 'int' [-W\n             implicit-int]\n     900         5 | int __declspec(thread) foo;\n     901           |     ^~~~~~~~~~\n>> 902     _configtest.c:11: error: expected '{' at end of input\n     903        11 | }\n     904           |\n     905     _configtest.c:11: warning: control reaches end of non-void functio\n             n [-Wreturn-type]\n     906        11 | }\n     907           |",
    "y/core/src/umath -Inumpy/core/src/npysort -I/p/vast1/koparasy/BUIL\n             D/spack_cluster_installer/quartz/py-warpx/spack_env_dir/linux-rhel\n             7-broadwell/gcc-9.3.1/python-3.9.9-3o76m4utiyrsvuxwx2ytcv7sd3g35si\n             6/include/python3.9 -c'\n     1235    gcc: _configtest.c\n     1236    _configtest.c: In function 'main':\n>> 1237    _configtest.c:8:12: error: 'HAVE_DECL_SIGNBIT' undeclared (first u\n             se in this function); did you mean 'HAVE_DECL_ISNAN'?\n     1238        8 |     (void) HAVE_DECL_SIGNBIT;\n     1239          |            ^~~~~~~~~~~~~~~~~\n     1240          |            HAVE_DECL_ISNAN\n     1241    _configtest.c:8:12: note: each undeclared identifier is reported o\n             nly once for each function it appears in",
    "1238        8 |     (void) HAVE_DECL_SIGNBIT;\n     1239          |            ^~~~~~~~~~~~~~~~~\n     1240          |            HAVE_DECL_ISNAN\n     1241    _configtest.c:8:12: note: each undeclared identifier is reported o\n             nly once for each function it appears in\n     1242    _configtest.c: In function 'main':\n>> 1243    _configtest.c:8:12: error: 'HAVE_DECL_SIGNBIT' undeclared (first u\n             se in this function); did you mean 'HAVE_DECL_ISNAN'?\n     1244        8 |     (void) HAVE_DECL_SIGNBIT;\n     1245          |            ^~~~~~~~~~~~~~~~~\n     1246          |            HAVE_DECL_ISNAN\n     1247    _configtest.c:8:12: note: each undeclared identifier is reported o\n             nly once for each function it appears in",
    "st.o -o _configtest\n     1829    /usr/tce/packages/gcc/gcc-9.3.1/rh/usr/bin/../libexec/gcc/x86_64-r\n             edhat-linux/9/ld: _configtest.o: in function `main':\n  >> 1830    /tmp/koparasy/spack-stage/spack-stage-py-numpy-1.14.1-2d2rt5cmgzkd\n             7ib5m3j7vvhqeeky7wau/spack-src/_configtest.c:6: undefined referenc\n             e to `exp'\n>> 1831    collect2: error: ld returned 1 exit status\n     1832    /usr/tce/packages/gcc/gcc-9.3.1/rh/usr/bin/../libexec/gcc/x86_64-r\n             edhat-linux/9/ld: _configtest.o: in function `main':\n  >> 1833    /tmp/koparasy/spack-stage/spack-stage-py-numpy-1.14.1-2d2rt5cmgzkd\n             7ib5m3j7vvhqeeky7wau/spack-src/_configtest.c:6: undefined referenc\n             e to `exp'\n  >> 1834    collect2: error: ld returned 1 exit status",
    ">> 1831    collect2: error: ld returned 1 exit status\n     1832    /usr/tce/packages/gcc/gcc-9.3.1/rh/usr/bin/../libexec/gcc/x86_64-r\n             edhat-linux/9/ld: _configtest.o: in function `main':\n  >> 1833    /tmp/koparasy/spack-stage/spack-stage-py-numpy-1.14.1-2d2rt5cmgzkd\n             7ib5m3j7vvhqeeky7wau/spack-src/_configtest.c:6: undefined referenc\n             e to `exp'\n>> 1834    collect2: error: ld returned 1 exit status\n     1835    failure.\n     1836    removing: _configtest.c _configtest.o\n     1837    C compiler: /p/vast1/koparasy/BUILD/head/spack/lib/spack/env/gcc/g\n             cc -Wno-unused-result -Wsign-compare -DNDEBUG -g -fwrapv -O3 -Wall\n              -fPIC -fPIC\n     1838",
    "ath -Ibuild/src.linux-x86_64-3.9/numpy/core/src/private -Ibuild/sr\n             c.linux-x86_64-3.9/numpy/core/src/npymath -Ibuild/src.linux-x86_64\n             -3.9/numpy/core/src/private -Ibuild/src.linux-x86_64-3.9/numpy/cor\n             e/src/npymath -c'\n     4423    gcc: numpy/random/mtrand/mtrand.c\n     4424    numpy/random/mtrand/mtrand.c: In function 'PyInit_mtrand':\n>> 4425    numpy/random/mtrand/mtrand.c:43644:33: error: 'PyTypeObject' {aka\n             'struct _typeobject'} has no member named 'tp_print'\n     4426    43644 |   __pyx_type_6mtrand_RandomState.tp_print = 0;\n     4427          |                                 ^\n     4428    numpy/random/mtrand/mtrand.c: In function '__Pyx_ParseOptionalKeyw\n             ords':\n     4429    numpy/random/mtrand/mtrand.c:45175:21: warning: '_PyUnicode_get_ws",
    "yrsvuxwx2ytcv7sd3g35si6/include/python3.9/cpython/unicodeobject.h:\n             446:26: note: declared here\n     4535      446 | static inline Py_ssize_t _PyUnicode_get_wstr_length(PyObje\n             ct *op) {\n     4536          |                          ^~~~~~~~~~~~~~~~~~~~~~~~~~\n     4537    numpy/random/mtrand/mtrand.c: In function '__Pyx__ExceptionSave':\n>> 4538    numpy/random/mtrand/mtrand.c:45472:21: error: 'PyThreadState' {aka\n              'struct _ts'} has no member named 'exc_type'; did you mean 'curex\n             c_type'?\n     4539    45472 |     *type = tstate->exc_type;\n     4540          |                     ^~~~~~~~\n     4541          |                     curexc_type\n  >> 4542    numpy/random/mtrand/mtrand.c:45473:22: error: 'PyThreadState' {aka",
    ">> 4538    numpy/random/mtrand/mtrand.c:45472:21: error: 'PyThreadState' {aka\n              'struct _ts'} has no member named 'exc_type'; did you mean 'curex\n             c_type'?\n     4539    45472 |     *type = tstate->exc_type;\n     4540          |                     ^~~~~~~~\n     4541          |                     curexc_type\n>> 4542    numpy/random/mtrand/mtrand.c:45473:22: error: 'PyThreadState' {aka\n              'struct _ts'} has no member named 'exc_value'; did you mean 'cure\n             xc_value'?\n     4543    45473 |     *value = tstate->exc_value;\n     4544          |                      ^~~~~~~~~\n     4545          |                      curexc_value\n  >> 4546    numpy/random/mtrand/mtrand.c:45474:19: error: 'PyThreadState' {aka",
    ">> 4542    numpy/random/mtrand/mtrand.c:45473:22: error: 'PyThreadState' {aka\n              'struct _ts'} has no member named 'exc_value'; did you mean 'cure\n             xc_value'?\n     4543    45473 |     *value = tstate->exc_value;\n     4544          |                      ^~~~~~~~~\n     4545          |                      curexc_value\n>> 4546    numpy/random/mtrand/mtrand.c:45474:19: error: 'PyThreadState' {aka\n              'struct _ts'} has no member named 'exc_traceback'; did you mean '\n             curexc_traceback'?\n     4547    45474 |     *tb = tstate->exc_traceback;\n     4548          |                   ^~~~~~~~~~~~~\n     4549          |                   curexc_traceback\n     4550    numpy/random/mtrand/mtrand.c: In function '__Pyx__ExceptionReset':",
    "'struct _ts'} has no member named 'exc_traceback'; did you mean '\n             curexc_traceback'?\n     4547    45474 |     *tb = tstate->exc_traceback;\n     4548          |                   ^~~~~~~~~~~~~\n     4549          |                   curexc_traceback\n     4550    numpy/random/mtrand/mtrand.c: In function '__Pyx__ExceptionReset':\n>> 4551    numpy/random/mtrand/mtrand.c:45481:24: error: 'PyThreadState' {aka\n              'struct _ts'} has no member named 'exc_type'; did you mean 'curex\n             c_type'?\n     4552    45481 |     tmp_type = tstate->exc_type;\n     4553          |                        ^~~~~~~~\n     4554          |                        curexc_type\n  >> 4555    numpy/random/mtrand/mtrand.c:45482:25: error: 'PyThreadState' {aka",
    ">> 4551    numpy/random/mtrand/mtrand.c:45481:24: error: 'PyThreadState' {aka\n              'struct _ts'} has no member named 'exc_type'; did you mean 'curex\n             c_type'?\n     4552    45481 |     tmp_type = tstate->exc_type;\n     4553          |                        ^~~~~~~~\n     4554          |                        curexc_type\n>> 4555    numpy/random/mtrand/mtrand.c:45482:25: error: 'PyThreadState' {aka\n              'struct _ts'} has no member named 'exc_value'; did you mean 'cure\n             xc_value'?\n     4556    45482 |     tmp_value = tstate->exc_value;\n     4557          |                         ^~~~~~~~~\n     4558          |                         curexc_value\n  >> 4559    numpy/random/mtrand/mtrand.c:45483:22: error: 'PyThreadState' {aka",
    ">> 4555    numpy/random/mtrand/mtrand.c:45482:25: error: 'PyThreadState' {aka\n              'struct _ts'} has no member named 'exc_value'; did you mean 'cure\n             xc_value'?\n     4556    45482 |     tmp_value = tstate->exc_value;\n     4557          |                         ^~~~~~~~~\n     4558          |                         curexc_value\n>> 4559    numpy/random/mtrand/mtrand.c:45483:22: error: 'PyThreadState' {aka\n              'struct _ts'} has no member named 'exc_traceback'; did you mean '\n             curexc_traceback'?\n     4560    45483 |     tmp_tb = tstate->exc_traceback;\n     4561          |                      ^~~~~~~~~~~~~\n     4562          |                      curexc_traceback\n  >> 4563    numpy/random/mtrand/mtrand.c:45484:13: error: 'PyThreadState' {aka",
    ">> 4559    numpy/random/mtrand/mtrand.c:45483:22: error: 'PyThreadState' {aka\n              'struct _ts'} has no member named 'exc_traceback'; did you mean '\n             curexc_traceback'?\n     4560    45483 |     tmp_tb = tstate->exc_traceback;\n     4561          |                      ^~~~~~~~~~~~~\n     4562          |                      curexc_traceback\n>> 4563    numpy/random/mtrand/mtrand.c:45484:13: error: 'PyThreadState' {aka\n              'struct _ts'} has no member named 'exc_type'; did you mean 'curex\n             c_type'?\n     4564    45484 |     tstate->exc_type = type;\n     4565          |             ^~~~~~~~\n     4566          |             curexc_type\n  >> 4567    numpy/random/mtrand/mtrand.c:45485:13: error: 'PyThreadState' {aka",
    ">> 4563    numpy/random/mtrand/mtrand.c:45484:13: error: 'PyThreadState' {aka\n              'struct _ts'} has no member named 'exc_type'; did you mean 'curex\n             c_type'?\n     4564    45484 |     tstate->exc_type = type;\n     4565          |             ^~~~~~~~\n     4566          |             curexc_type\n>> 4567    numpy/random/mtrand/mtrand.c:45485:13: error: 'PyThreadState' {aka\n              'struct _ts'} has no member named 'exc_value'; did you mean 'cure\n             xc_value'?\n     4568    45485 |     tstate->exc_value = value;\n     4569          |             ^~~~~~~~~\n     4570          |             curexc_value\n  >> 4571    numpy/random/mtrand/mtrand.c:45486:13: error: 'PyThreadState' {aka",
    ">> 4567    numpy/random/mtrand/mtrand.c:45485:13: error: 'PyThreadState' {aka\n              'struct _ts'} has no member named 'exc_value'; did you mean 'cure\n             xc_value'?\n     4568    45485 |     tstate->exc_value = value;\n     4569          |             ^~~~~~~~~\n     4570          |             curexc_value\n>> 4571    numpy/random/mtrand/mtrand.c:45486:13: error: 'PyThreadState' {aka\n              'struct _ts'} has no member named 'exc_traceback'; did you mean '\n             curexc_traceback'?\n     4572    45486 |     tstate->exc_traceback = tb;\n     4573          |             ^~~~~~~~~~~~~\n     4574          |             curexc_traceback\n     4575    numpy/random/mtrand/mtrand.c: In function '__Pyx__GetException':",
    "'struct _ts'} has no member named 'exc_traceback'; did you mean '\n             curexc_traceback'?\n     4572    45486 |     tstate->exc_traceback = tb;\n     4573          |             ^~~~~~~~~~~~~\n     4574          |             curexc_traceback\n     4575    numpy/random/mtrand/mtrand.c: In function '__Pyx__GetException':\n>> 4576    numpy/random/mtrand/mtrand.c:45541:24: error: 'PyThreadState' {aka\n              'struct _ts'} has no member named 'exc_type'; did you mean 'curex\n             c_type'?\n     4577    45541 |     tmp_type = tstate->exc_type;\n     4578          |                        ^~~~~~~~\n     4579          |                        curexc_type\n  >> 4580    numpy/random/mtrand/mtrand.c:45542:25: error: 'PyThreadState' {aka",
    ">> 4576    numpy/random/mtrand/mtrand.c:45541:24: error: 'PyThreadState' {aka\n              'struct _ts'} has no member named 'exc_type'; did you mean 'curex\n             c_type'?\n     4577    45541 |     tmp_type = tstate->exc_type;\n     4578          |                        ^~~~~~~~\n     4579          |                        curexc_type\n>> 4580    numpy/random/mtrand/mtrand.c:45542:25: error: 'PyThreadState' {aka\n              'struct _ts'} has no member named 'exc_value'; did you mean 'cure\n             xc_value'?\n     4581    45542 |     tmp_value = tstate->exc_value;\n     4582          |                         ^~~~~~~~~\n     4583          |                         curexc_value\n  >> 4584    numpy/random/mtrand/mtrand.c:45543:22: error: 'PyThreadState' {aka",
    ">> 4580    numpy/random/mtrand/mtrand.c:45542:25: error: 'PyThreadState' {aka\n              'struct _ts'} has no member named 'exc_value'; did you mean 'cure\n             xc_value'?\n     4581    45542 |     tmp_value = tstate->exc_value;\n     4582          |                         ^~~~~~~~~\n     4583          |                         curexc_value\n>> 4584    numpy/random/mtrand/mtrand.c:45543:22: error: 'PyThreadState' {aka\n              'struct _ts'} has no member named 'exc_traceback'; did you mean '\n             curexc_traceback'?\n     4585    45543 |     tmp_tb = tstate->exc_traceback;\n     4586          |                      ^~~~~~~~~~~~~\n     4587          |                      curexc_traceback\n  >> 4588    numpy/random/mtrand/mtrand.c:45544:13: error: 'PyThreadState' {aka",
    ">> 4584    numpy/random/mtrand/mtrand.c:45543:22: error: 'PyThreadState' {aka\n              'struct _ts'} has no member named 'exc_traceback'; did you mean '\n             curexc_traceback'?\n     4585    45543 |     tmp_tb = tstate->exc_traceback;\n     4586          |                      ^~~~~~~~~~~~~\n     4587          |                      curexc_traceback\n>> 4588    numpy/random/mtrand/mtrand.c:45544:13: error: 'PyThreadState' {aka\n              'struct _ts'} has no member named 'exc_type'; did you mean 'curex\n             c_type'?\n     4589    45544 |     tstate->exc_type = local_type;\n     4590          |             ^~~~~~~~\n     4591          |             curexc_type\n  >> 4592    numpy/random/mtrand/mtrand.c:45545:13: error: 'PyThreadState' {aka",
    ">> 4588    numpy/random/mtrand/mtrand.c:45544:13: error: 'PyThreadState' {aka\n              'struct _ts'} has no member named 'exc_type'; did you mean 'curex\n             c_type'?\n     4589    45544 |     tstate->exc_type = local_type;\n     4590          |             ^~~~~~~~\n     4591          |             curexc_type\n>> 4592    numpy/random/mtrand/mtrand.c:45545:13: error: 'PyThreadState' {aka\n              'struct _ts'} has no member named 'exc_value'; did you mean 'cure\n             xc_value'?\n     4593    45545 |     tstate->exc_value = local_value;\n     4594          |             ^~~~~~~~~\n     4595          |             curexc_value\n  >> 4596    numpy/random/mtrand/mtrand.c:45546:13: error: 'PyThreadState' {aka",
    ">> 4592    numpy/random/mtrand/mtrand.c:45545:13: error: 'PyThreadState' {aka\n              'struct _ts'} has no member named 'exc_value'; did you mean 'cure\n             xc_value'?\n     4593    45545 |     tstate->exc_value = local_value;\n     4594          |             ^~~~~~~~~\n     4595          |             curexc_value\n>> 4596    numpy/random/mtrand/mtrand.c:45546:13: error: 'PyThreadState' {aka\n              'struct _ts'} has no member named 'exc_traceback'; did you mean '\n             curexc_traceback'?\n     4597    45546 |     tstate->exc_traceback = local_tb;\n     4598          |             ^~~~~~~~~~~~~\n     4599          |             curexc_traceback\n     4600    numpy/random/mtrand/mtrand.c: In function 'PyInit_mtrand':",
    "'struct _ts'} has no member named 'exc_traceback'; did you mean '\n             curexc_traceback'?\n     4597    45546 |     tstate->exc_traceback = local_tb;\n     4598          |             ^~~~~~~~~~~~~\n     4599          |             curexc_traceback\n     4600    numpy/random/mtrand/mtrand.c: In function 'PyInit_mtrand':\n>> 4601    numpy/random/mtrand/mtrand.c:43644:33: error: 'PyTypeObject' {aka\n             'struct _typeobject'} has no member named 'tp_print'\n     4602    43644 |   __pyx_type_6mtrand_RandomState.tp_print = 0;\n     4603          |                                 ^\n     4604    numpy/random/mtrand/mtrand.c: In function '__Pyx_ParseOptionalKeyw\n             ords':\n     4605    numpy/random/mtrand/mtrand.c:45175:21: warning: '_PyUnicode_get_ws",
    "yrsvuxwx2ytcv7sd3g35si6/include/python3.9/cpython/unicodeobject.h:\n             446:26: note: declared here\n     4711      446 | static inline Py_ssize_t _PyUnicode_get_wstr_length(PyObje\n             ct *op) {\n     4712          |                          ^~~~~~~~~~~~~~~~~~~~~~~~~~\n     4713    numpy/random/mtrand/mtrand.c: In function '__Pyx__ExceptionSave':\n>> 4714    numpy/random/mtrand/mtrand.c:45472:21: error: 'PyThreadState' {aka\n              'struct _ts'} has no member named 'exc_type'; did you mean 'curex\n             c_type'?\n     4715    45472 |     *type = tstate->exc_type;\n     4716          |                     ^~~~~~~~\n     4717          |                     curexc_type\n  >> 4718    numpy/random/mtrand/mtrand.c:45473:22: error: 'PyThreadState' {aka",
    ">> 4714    numpy/random/mtrand/mtrand.c:45472:21: error: 'PyThreadState' {aka\n              'struct _ts'} has no member named 'exc_type'; did you mean 'curex\n             c_type'?\n     4715    45472 |     *type = tstate->exc_type;\n     4716          |                     ^~~~~~~~\n     4717          |                     curexc_type\n>> 4718    numpy/random/mtrand/mtrand.c:45473:22: error: 'PyThreadState' {aka\n              'struct _ts'} has no member named 'exc_value'; did you mean 'cure\n             xc_value'?\n     4719    45473 |     *value = tstate->exc_value;\n     4720          |                      ^~~~~~~~~\n     4721          |                      curexc_value\n  >> 4722    numpy/random/mtrand/mtrand.c:45474:19: error: 'PyThreadState' {aka",
    ">> 4718    numpy/random/mtrand/mtrand.c:45473:22: error: 'PyThreadState' {aka\n              'struct _ts'} has no member named 'exc_value'; did you mean 'cure\n             xc_value'?\n     4719    45473 |     *value = tstate->exc_value;\n     4720          |                      ^~~~~~~~~\n     4721          |                      curexc_value\n>> 4722    numpy/random/mtrand/mtrand.c:45474:19: error: 'PyThreadState' {aka\n              'struct _ts'} has no member named 'exc_traceback'; did you mean '\n             curexc_traceback'?\n     4723    45474 |     *tb = tstate->exc_traceback;\n     4724          |                   ^~~~~~~~~~~~~\n     4725          |                   curexc_traceback\n     4726    numpy/random/mtrand/mtrand.c: In function '__Pyx__ExceptionReset':",
    "'struct _ts'} has no member named 'exc_traceback'; did you mean '\n             curexc_traceback'?\n     4723    45474 |     *tb = tstate->exc_traceback;\n     4724          |                   ^~~~~~~~~~~~~\n     4725          |                   curexc_traceback\n     4726    numpy/random/mtrand/mtrand.c: In function '__Pyx__ExceptionReset':\n>> 4727    numpy/random/mtrand/mtrand.c:45481:24: error: 'PyThreadState' {aka\n              'struct _ts'} has no member named 'exc_type'; did you mean 'curex\n             c_type'?\n     4728    45481 |     tmp_type = tstate->exc_type;\n     4729          |                        ^~~~~~~~\n     4730          |                        curexc_type\n  >> 4731    numpy/random/mtrand/mtrand.c:45482:25: error: 'PyThreadState' {aka",
    ">> 4727    numpy/random/mtrand/mtrand.c:45481:24: error: 'PyThreadState' {aka\n              'struct _ts'} has no member named 'exc_type'; did you mean 'curex\n             c_type'?\n     4728    45481 |     tmp_type = tstate->exc_type;\n     4729          |                        ^~~~~~~~\n     4730          |                        curexc_type\n>> 4731    numpy/random/mtrand/mtrand.c:45482:25: error: 'PyThreadState' {aka\n              'struct _ts'} has no member named 'exc_value'; did you mean 'cure\n             xc_value'?\n     4732    45482 |     tmp_value = tstate->exc_value;\n     4733          |                         ^~~~~~~~~\n     4734          |                         curexc_value\n  >> 4735    numpy/random/mtrand/mtrand.c:45483:22: error: 'PyThreadState' {aka",
    ">> 4731    numpy/random/mtrand/mtrand.c:45482:25: error: 'PyThreadState' {aka\n              'struct _ts'} has no member named 'exc_value'; did you mean 'cure\n             xc_value'?\n     4732    45482 |     tmp_value = tstate->exc_value;\n     4733          |                         ^~~~~~~~~\n     4734          |                         curexc_value\n>> 4735    numpy/random/mtrand/mtrand.c:45483:22: error: 'PyThreadState' {aka\n              'struct _ts'} has no member named 'exc_traceback'; did you mean '\n             curexc_traceback'?\n     4736    45483 |     tmp_tb = tstate->exc_traceback;\n     4737          |                      ^~~~~~~~~~~~~\n     4738          |                      curexc_traceback\n  >> 4739    numpy/random/mtrand/mtrand.c:45484:13: error: 'PyThreadState' {aka",
    ">> 4735    numpy/random/mtrand/mtrand.c:45483:22: error: 'PyThreadState' {aka\n              'struct _ts'} has no member named 'exc_traceback'; did you mean '\n             curexc_traceback'?\n     4736    45483 |     tmp_tb = tstate->exc_traceback;\n     4737          |                      ^~~~~~~~~~~~~\n     4738          |                      curexc_traceback\n>> 4739    numpy/random/mtrand/mtrand.c:45484:13: error: 'PyThreadState' {aka\n              'struct _ts'} has no member named 'exc_type'; did you mean 'curex\n             c_type'?\n     4740    45484 |     tstate->exc_type = type;\n     4741          |             ^~~~~~~~\n     4742          |             curexc_type\n  >> 4743    numpy/random/mtrand/mtrand.c:45485:13: error: 'PyThreadState' {aka",
    ">> 4739    numpy/random/mtrand/mtrand.c:45484:13: error: 'PyThreadState' {aka\n              'struct _ts'} has no member named 'exc_type'; did you mean 'curex\n             c_type'?\n     4740    45484 |     tstate->exc_type = type;\n     4741          |             ^~~~~~~~\n     4742          |             curexc_type\n>> 4743    numpy/random/mtrand/mtrand.c:45485:13: error: 'PyThreadState' {aka\n              'struct _ts'} has no member named 'exc_value'; did you mean 'cure\n             xc_value'?\n     4744    45485 |     tstate->exc_value = value;\n     4745          |             ^~~~~~~~~\n     4746          |             curexc_value\n  >> 4747    numpy/random/mtrand/mtrand.c:45486:13: error: 'PyThreadState' {aka",
    ">> 4743    numpy/random/mtrand/mtrand.c:45485:13: error: 'PyThreadState' {aka\n              'struct _ts'} has no member named 'exc_value'; did you mean 'cure\n             xc_value'?\n     4744    45485 |     tstate->exc_value = value;\n     4745          |             ^~~~~~~~~\n     4746          |             curexc_value\n>> 4747    numpy/random/mtrand/mtrand.c:45486:13: error: 'PyThreadState' {aka\n              'struct _ts'} has no member named 'exc_traceback'; did you mean '\n             curexc_traceback'?\n     4748    45486 |     tstate->exc_traceback = tb;\n     4749          |             ^~~~~~~~~~~~~\n     4750          |             curexc_traceback\n     4751    numpy/random/mtrand/mtrand.c: In function '__Pyx__GetException':",
    "'struct _ts'} has no member named 'exc_traceback'; did you mean '\n             curexc_traceback'?\n     4748    45486 |     tstate->exc_traceback = tb;\n     4749          |             ^~~~~~~~~~~~~\n     4750          |             curexc_traceback\n     4751    numpy/random/mtrand/mtrand.c: In function '__Pyx__GetException':\n>> 4752    numpy/random/mtrand/mtrand.c:45541:24: error: 'PyThreadState' {aka\n              'struct _ts'} has no member named 'exc_type'; did you mean 'curex\n             c_type'?\n     4753    45541 |     tmp_type = tstate->exc_type;\n     4754          |                        ^~~~~~~~\n     4755          |                        curexc_type\n  >> 4756    numpy/random/mtrand/mtrand.c:45542:25: error: 'PyThreadState' {aka",
    ">> 4752    numpy/random/mtrand/mtrand.c:45541:24: error: 'PyThreadState' {aka\n              'struct _ts'} has no member named 'exc_type'; did you mean 'curex\n             c_type'?\n     4753    45541 |     tmp_type = tstate->exc_type;\n     4754          |                        ^~~~~~~~\n     4755          |                        curexc_type\n>> 4756    numpy/random/mtrand/mtrand.c:45542:25: error: 'PyThreadState' {aka\n              'struct _ts'} has no member named 'exc_value'; did you mean 'cure\n             xc_value'?\n     4757    45542 |     tmp_value = tstate->exc_value;\n     4758          |                         ^~~~~~~~~\n     4759          |                         curexc_value\n  >> 4760    numpy/random/mtrand/mtrand.c:45543:22: error: 'PyThreadState' {aka",
    ">> 4756    numpy/random/mtrand/mtrand.c:45542:25: error: 'PyThreadState' {aka\n              'struct _ts'} has no member named 'exc_value'; did you mean 'cure\n             xc_value'?\n     4757    45542 |     tmp_value = tstate->exc_value;\n     4758          |                         ^~~~~~~~~\n     4759          |                         curexc_value\n>> 4760    numpy/random/mtrand/mtrand.c:45543:22: error: 'PyThreadState' {aka\n              'struct _ts'} has no member named 'exc_traceback'; did you mean '\n             curexc_traceback'?\n     4761    45543 |     tmp_tb = tstate->exc_traceback;\n     4762          |                      ^~~~~~~~~~~~~\n     4763          |                      curexc_traceback\n  >> 4764    numpy/random/mtrand/mtrand.c:45544:13: error: 'PyThreadState' {aka",
    ">> 4760    numpy/random/mtrand/mtrand.c:45543:22: error: 'PyThreadState' {aka\n              'struct _ts'} has no member named 'exc_traceback'; did you mean '\n             curexc_traceback'?\n     4761    45543 |     tmp_tb = tstate->exc_traceback;\n     4762          |                      ^~~~~~~~~~~~~\n     4763          |                      curexc_traceback\n>> 4764    numpy/random/mtrand/mtrand.c:45544:13: error: 'PyThreadState' {aka\n              'struct _ts'} has no member named 'exc_type'; did you mean 'curex\n             c_type'?\n     4765    45544 |     tstate->exc_type = local_type;\n     4766          |             ^~~~~~~~\n     4767          |             curexc_type\n  >> 4768    numpy/random/mtrand/mtrand.c:45545:13: error: 'PyThreadState' {aka",
    ">> 4764    numpy/random/mtrand/mtrand.c:45544:13: error: 'PyThreadState' {aka\n              'struct _ts'} has no member named 'exc_type'; did you mean 'curex\n             c_type'?\n     4765    45544 |     tstate->exc_type = local_type;\n     4766          |             ^~~~~~~~\n     4767          |             curexc_type\n>> 4768    numpy/random/mtrand/mtrand.c:45545:13: error: 'PyThreadState' {aka\n              'struct _ts'} has no member named 'exc_value'; did you mean 'cure\n             xc_value'?\n     4769    45545 |     tstate->exc_value = local_value;\n     4770          |             ^~~~~~~~~\n     4771          |             curexc_value\n  >> 4772    numpy/random/mtrand/mtrand.c:45546:13: error: 'PyThreadState' {aka",
    ">> 4768    numpy/random/mtrand/mtrand.c:45545:13: error: 'PyThreadState' {aka\n              'struct _ts'} has no member named 'exc_value'; did you mean 'cure\n             xc_value'?\n     4769    45545 |     tstate->exc_value = local_value;\n     4770          |             ^~~~~~~~~\n     4771          |             curexc_value\n>> 4772    numpy/random/mtrand/mtrand.c:45546:13: error: 'PyThreadState' {aka\n              'struct _ts'} has no member named 'exc_traceback'; did you mean '\n             curexc_traceback'?\n     4773    45546 |     tstate->exc_traceback = local_tb;\n     4774          |             ^~~~~~~~~~~~~\n     4775          |             curexc_traceback\n  >> 4776    error: Command \"/p/vast1/koparasy/BUILD/head/spack/lib/spack/env/g"
]